\picturechapter{In-cell structures of conserved supramolecular protein arrays at the mitochondria-cytoskeleton interface in mammalian sperm}{Chapter_covers/chapter_cover_4.pdf} \label{ch-4}
\vspace*{0.25cm}

\footnotesize Miguel Ricardo Leung, Riccardo Zenezini-Chiozzi, Marc C. Roelofs, Johannes F. Hevler, Ravi Teja Ravi, Paula Maitan, Min Zhang, Heiko Henning, Elizabeth G. Bromfield, Stuart C. Howes, Bart M. Gadella, Albert J.R. Heck, and Tzviya Zeev-Ben-Mordehai
%
\begin{center}
	\vspace{1.5cm}
	\includegraphics[width=0.65\textwidth]{Chapter.4/Figures/Chapter_cover.png}
	\vspace{0.5cm}
\end{center}
%
\begin{flushleft}
	\vspace*{\fill}
	\rule{\textwidth}{1pt}\\[0cm]
	\textbf{This chapter is based on work in the following publication:}\\
	\footnotesize
	\textbf{\emph{PNAS}} (2021), 118:e2110996118, doi:10.1073/pnas.2110996118\\
\end{flushleft}
%%Additional footnote to potentially add for clarification
%{\footnotesize The manuscript resulted from a collaboration which was led by Miguel Ricardo Leung and Tzviya Zeev-Ben-Mordehai. We contributed all proteomics and cross-linking mass spectrometry experiments and respective data analysis.}
\begin{abstract102}
	Mitochondria-cytoskeleton interactions modulate cellular physiology by regulating mitochondrial transport, positioning, and immobilization. However, there is very little structural information defining mitochondria-cytoskeleton interfaces in any cell type. Here, we use cryo-focused ion beam milling-enabled cryo-electron tomography to image mammalian sperm, where mitochondria wrap around the flagellar  cytoskeleton. We find that mitochondria are tethered to their neighbors through intermitochondrial linkers and are anchored to the cytoskeleton through ordered arrays on the outer mitochondrial membrane. We use subtomogram averaging to resolve in-cell structures of these arrays from three mammalian species, revealing they are conserved across species despite variations in mitochondrial dimensions and cristae organization. We find that the arrays consist of boat-shaped particles anchored on a network of membrane pores whose arrangement and dimensions are consistent with voltage dependent anion channels. Proteomics and in-cell cross-linking mass spectrometry suggest that the conserved arrays are composed of glycerol kinase-like proteins. Ordered supramolecular assemblies may serve to stabilize similar contact sites in other cell types where mitochondria need to be immobilized in specific subcellular environments, such as in muscles and neurons.
\end{abstract102}
%%Main Text
\section{Introduction}
\lettrine[lraise=0.1, nindent=0em, slope=-.5em]{I}{n} many cell types, mitochondria collectively form a dynamic network whose members divide, fuse, and communicate with one another \cite{Glancy2015, Viana2020, Vincent2017}. Through interactions with the cytoskeleton, mitochondria are transported - sometimes across large distances - and positioned in response to dynamic stimuli \cite{Fenton2021, Moore2018}. Interactions with the cytoskeleton can also restrain mitochondria to specific subcellular locations. In neurons, axonal mitochondria can be immobilized by interactions with the microtubule or actin cytoskeletons \cite{Chen2013, Gutnick2019, Kang2008}. In cardiac and skeletal muscle, mitochondrial distribution is regulated by interactions with myofibrils and  intermediate filaments \cite{Milner2000, Stone2007}. However, despite the prevalence of intermitochondria and mitochondria-cytoskeleton interactions and their integral roles in cellular function, there is very little information on the molecular architectures of these interaction sites in any cell type.

One of the most striking mitochondrial configurations occurs in amniote sperm, where mitochondria are arranged in a spiral around the axoneme, defining a region called the midpiece \cite{Fawcett1970, Fawcett1975}. Mitochondria are among the few organelles retained in sperm throughout their maturation, during which they otherwise lose most of their cytoplasm and organelles \emph{en route} to becoming highly streamlined cells specialized for finding and fusing with the egg. The extensive mitochondrial sheath in amniote sperm may be an adaptation needed to power the large, long flagellum in these lineages. Variations in midpiece morphometry affect sperm motility and competitiveness \cite{Firman2010, Fisher2016}, and different species rely on energy from mitochondrial respiration to different extents \cite{Tourmente2015, Marin2003}, warranting comparative studies of mitochondrial structure across species.

The core of the midpiece is the flagellar  cytoskeleton, composed of the microtubule-based axoneme and accessory elements called outer dense fibers (ODFs). A poorly-characterized network of cytoskeletal filaments called the submitochondrial reticulum lies between the ODFs and the mitochondria. The submitochondrial reticulum co-purifies with the outer mitochondrial membrane (OMM), suggesting that they are intimately associated \cite{Olson1986, Olson1990}. Mitochondria wrap around the cytoskeleton and are in turn ensheathed by the plasma membrane. As a consequence of this arrangement, each mitochondrion has three distinct surfaces \cite{Olson1992} - one facing the axoneme, one facing the plasma membrane, and one facing neighboring mitochondria. Thin-section electron microscopy (EM) \cite{Olson1992} and freeze-fracture EM \cite{Friend1981, Woolley2005} revealed that each surface is characterized by a unique membrane protein profile. In particular, these studies uncovered an ordered array of particles on the axoneme-facing surface of sperm mitochondria. Notwithstanding the insight gained from these methods, such techniques require harsh sample preparation steps that can distort fine cellular structure and limit achievable resolution \cite{Al-Amoudi2004}. As such, the molecular landscape of intermitochondrial and mitochondrial cytoskeleton contacts in the sperm midpiece remains largely unexplored.

Assembly of the mitochondrial sheath occurs late in spermiogenesis and involves an intricately choreographed series of events \cite{Otani1988, Ho2007}. Initially, spherical mitochondria are broadly distributed in the cytoplasm. Mitochondria are then recruited to the flagellum, where they form ordered rows along the flagellar axis. Finally, mitochondria elongate and twist around the axoneme. While our understanding of the molecular details of these processes is cursory at best, studies on gene-disrupted mice have implicated several proteins in mitochondrial sheath morphogenesis. For instance, mice expressing mutant forms of kinesin light chain 3 (KLC3) have malformed midpieces, hinting at a role for microtubule-based transport \cite{Zhang2012}. Other examples are the voltage dependent anion channels (VDACs), which are highly abundant mitochondrial proteins that mediate transport of metabolites, ions, and nucleotides like ATP across the OMM \cite{Colombini2012}. Male mice lacking VDAC3 are infertile and their sperm cells have disorganized mitochondrial sheaths \cite{Sampson2001}, so VDACs may also have unappreciated roles in mitochondrial trafficking; indeed, KLC3 binds mitochondria through VDAC2 \cite{Zhang2012}. Similarly, disrupting sperm-specific isoforms of glycerol kinase leads to gaps in the mitochondrial sheath despite proper initial alignment of spherical mitochondria \cite{Chen2017a, Shimada2019}. Mice lacking spermatogenesis-associated protein 19 (SPATA19) \cite{Mi2015}, glutathione peroxidase 4 (GPX4) \cite{Schneider2009, Imai2009} also have structurally abnormal mitochondria.\\[0.25cm]
\raggedbottom
Here, we use cryo-focused ion beam (cryo-FIB) milling-enabled cryo-electron tomography (cryo-ET) to image the mitochondrial sheath in mature sperm from three mammalian species. We take advantage of the uniquely multi-scale capabilities of cryo-ET to unveil new aspects both of the overall organization of the mitochondrial sheath and of the molecular structures important for its assembly. We find that mitochondria are tethered to their neighbors through intermitochondrial linkers and to the underlying cytoskeleton through conserved protein arrays on the OMM. These arrays were first described by deep-etch freeze-fracture EM in guinea pig sperm \cite{Friend1981}. Here, we resolve the three-dimensional structures of these arrays in a near-native state and at molecular resolution, revealing how they anchor onto the mitochondrial membrane and how they interact with the flagellar  cytoskeleton. Subtomogram averaging reveals that the arrays consist of two-fold-symmetric boat-shaped particles anchored on a lattice of OMM pores whose arrangement and dimensions are consistent with VDACs. Proteomics and in-cell cross-linking mass spectrometry suggest that the arrays consist of glycerol kinase (GK)-like proteins. Our data thus show that although mitochondrial dimensions and cristae architecture vary across species, the architecture of the mitochondria-cytoskeleton interface is conserved at the molecular level.
\pagebreak
%%Results
\section{Results}
\subsection*{Mitochondrial dimensions and cristae organization vary across\linebreak species}
We imaged the mitochondrial sheath in mature sperm from three mammalian species, namely the pig \emph{Sus scrofa}, the horse \emph{Equus caballus}, and the mouse \emph{Mus musculus} (\textbf{\autoref{fig:ch4_fig1}}). These species differ in terms of sperm size, motility patterns, and metabolism. To visualize the overall organization of the mitochondrial sheath, we imaged whole sperm with a Volta phase plate (VPP) \cite{Fukuda2015, Danev2014}. Neural-network based segmentation \cite{Chen2017b} of the mitochondrial membrane allowed us to visualize mitochondrial organization in three dimensions (\textbf{\autoref{fig:ch4_fig1}A-F}). To investigate variations in mitochondrial width along the midpiece, we first measured the width of each mitochondrion at multiple points along its length. We then divided mitochondria into groups based on their positions along the midpiece, as measured by their distance from the head (\textbf{\autoref{fig:ch4_app_fig1}}). The midpiece is $\sim$10 $\mu$m long in both pig and horse sperm, but $\sim$20 $\mu$m long in mouse sperm, so each group represents $\sim$2 $\mu$m in the pig and the horse and $\sim$4 $\mu$m in the mouse. We found that mouse sperm mitochondria are $\sim$1.5 times wider than pig and horse sperm mitochondria overall (\textbf{\autoref{fig:ch4_app_fig1}A}). In all three species studied, most mitochondria in the middle ($\sim$60\%) of the midpiece are crescent-shaped tubes (\textbf{\autoref{fig:ch4_fig1}D-F}) with consistent widths along their lengths (\textbf{\autoref{fig:ch4_app_fig1}B}). Mitochondria at the proximal end of the midpiece are larger than their more distal counterparts (\textbf{\autoref{fig:ch4_fig1}A-C} and \textbf{\autoref{fig:ch4_app_fig1}A}). Moreover, proximal mitochondria have more variable shapes, evidenced by greater variation in their widths at different point along their lengths (\textbf{\autoref{fig:ch4_app_fig1}B}). Because mitochondria wrap around the axoneme, variations in mitochondrial dimensions both across species and along the proximodistal axis of the flagellum affect the overall diameter and rigidity of the midpiece, likely fine-tuning the hydrodynamics of sperm motility.
\begin{figure*}[t]
	\center
	\includegraphics[]{Chapter.4/Figures/Figure1.png}
	\caption{\textbf{Mitochondrial dimensions and cristae organization vary across species.} \textbf{A-F.} Slices through Volta phase plate cryo-tomograms (left) and corresponding three-dimensional segmentations (right) of mitochondria from the start \textbf{A-C.} or middle \textbf{D-F.} of the midpiece from pig \textbf{A,D.}, horse \textbf{B,E.}, and mouse \textbf{C,F.} sperm. \textbf{G-L.} Slices through cryo-tomograms of FIB-milled pig \textbf{G,H.}, horse \textbf{I,J.}, and mouse \textbf{K,L.} sperm midpieces. Right panels show digital zooms of the regions boxed out in the left panels. The outer mitochondrial membrane is traced in green, the inner mitochondrial membrane in yellow, and the plasma membrane in blue. Arrowheads indicate inter-mitochondrial linker complexes. Labels: nuc - nucleus, sc - segmented columns, m - mitochondria, odf - outer dense fibers, dc - distal centriole, ax - axoneme, mtd - microtubule doublets, cpa - central pair apparatus, pm - plasma membrane. Scale bars: \textbf{A-L.} left panels - 250 nm, \textbf{G-L.} right panels - 100 nm.}
	\label{fig:ch4_fig1}
\end{figure*}
To visualize the internal organization of sperm mitochondria in a near-native state, we imaged sperm thinned by cryo-FIB milling (\textbf{\autoref{fig:ch4_fig1}G-L}). This revealed unexpected diversity in the internal ultrastructure of mitochondria across mammalian species, especially in terms of cristae morphology. Horse sperm mitochondria have an expanded intermembrane space and a condensed matrix (\textbf{\autoref{fig:ch4_fig1}I-J}). Mouse sperm mitochondria have an expanded matrix, with a narrow intermembrane space and thin cristae (\textbf{\autoref{fig:ch4_fig1}K-L}). Pig sperm mitochondrial morphology is intermediate (\textbf{\autoref{fig:ch4_fig1}G-H}), and although the mitochondrial matrix was dense, we could identify individual complexes that resembled ATP synthase on cristae of FIB-milled mitochondria (\textbf{\autoref{fig:ch4_app_fig2}A-B}), which was confirmed by subtomogram averaging (\textbf{\autoref{fig:ch4_app_fig2}B'}).

Inter-species differences in cristae morphology correlate with measurements of matrix volume relative to mitochondrial volume (\textbf{\autoref{fig:ch4_app_fig2}D}). In this regard, horse sperm mitochondria resemble “condensed” mitochondria, which correlate with higher rates of oxidative activity in a number of different cell types, including developing germ cells, neurons, and liver \cite{DeMartino1979, Perkins2011, Hackenbrock1968}. Indeed, horse sperm are dependent on oxidative phosphorylation \cite{Davila2016}, whereas pig \cite{Marin2003} and mouse sperm \cite{Mukai2004, Odet2013} are thought to rely largely on a glycolytic mechanisms.%\raggedbottom
%
\subsection*{Intermitochondrial junctions are associated with linker complexes}
Mitochondria are closely packed within the mitochondrial sheath, but it is unclear whether or how individual organelles communicate with their neighbors. To address this, we imaged intermitochondrial junctions captured in FIB-milled sperm lamellae. We observed trans-mitochondrial cristae alignment in mouse sperm\linebreak (\textbf{\autoref{fig:ch4_fig1}K-L}) (44/91 junctions across 14 tomograms), but not in pig or in horse sperm (\textbf{\autoref{fig:ch4_fig1}G-J}) (0/67 junctions across 11 tomograms and 0/41 junctions across 8 tomograms respectively). Trans-mitochondrial cristae alignment has also been observed in muscle tissue of various organisms, and is proposed to mediate electrochemical coupling between adjacent mitochondria \cite{Picard2015}. To our knowledge, this is the first time this phenomenon has been observed in mature sperm from any lineage. It is particularly curious, however, that trans-mitochondrial cristae alignment in sperm is species-specific.

We found that intermitochondrial junctions are characterized by novel intermitochondrial linker complexes in all three species (arrowheads in \textbf{\autoref{fig:ch4_fig1}G-L}, \textbf{\autoref{fig:ch4_app_fig2}C}). These intermitochondrial linkers span the $\sim$8-nm distance between the outer membranes of neighboring mitochondria. In mouse sperm, these linkers are specifically associated with sites of trans-mitochondrial cristae alignment\linebreak (\textbf{\autoref{fig:ch4_fig1}K-L}); in the pig and in the horse, they are positioned at regularly spaced intervals along intermitochondrial junctions (\textbf{\autoref{fig:ch4_fig1}I,J}). Electron-dense intermitochondrial junctions were also seen in cardiomyocytes by classical EM \cite{Huang2013, Picard2015, Duvert1985}. Thus, it is plausible that the as-yet-unidentified linker complexes we visualize here represent a more general structural mechanism for orchestrating intermitochondrial communication in various cell types.
%
\subsection*{Ordered protein arrays at the mitochondria-cytoskeleton interface are conserved across species}
To determine how mitochondria interact with the flagellar cytoskeleton, we imaged the mitochondria-cytoskeleton interface in cryo-FIB milled lamellae (\textbf{\autoref{fig:ch4_fig2}}). Previous studies using freeze-fracture EM on guinea pig sperm \cite{Friend1981} and mouse sperm \cite{Woolley2005}, and thin-section EM on golden hamster sperm \cite{Olson1992} uncovered an ordered array of particles arranged in a ladder-like pattern on the axoneme-facing surface of sperm mitochondria. We found these arrays on the axoneme-facing surface of the OMM in our tomograms as well (\textbf{\autoref{fig:ch4_fig2}A}, yellow box), across all three species and along the entire midpiece (\textbf{\autoref{fig:ch4_app_fig3}A-F}). We observed direct interactions between the arrays and either the ODFs or the cytoskeletal filaments surrounding the ODFs (\textbf{\autoref{fig:ch4_fig2}B-C}, \textbf{\autoref{fig:ch4_app_fig4}A-L}), indicating that these arrays tether mitochondria to the midpiece cytoskeleton. The highly-ordered arrays are absent from the plasma membrane-surface (\textbf{\autoref{fig:ch4_fig2}A}, red box), which is consistent with freeze-fracture EM studies that detect only very short rows scattered randomly across this surface \cite{Friend1981}.

We then aligned and averaged sub-volumes containing the protein arrays and the underlying OMM from the axoneme-facing surface (\textbf{\autoref{fig:ch4_fig3}}, \textbf{\autoref{tab:ch4_app_tab1}}). Our averages revealed $\sim$22-nm-long two-fold-symmetric boat-shaped structures connected via four densities to a porous membrane (\textbf{\autoref{fig:ch4_fig3}}, \textbf{\autoref{fig:ch4_app_fig3}G-I}).
\begin{figure*}[bt!]
	\center
	\includegraphics[]{Chapter.4/Figures/Figure2.png}
	\caption{\textbf{Ordered protein arrays on the outer mitochondrial membrane interact with the cytoskeleton.} \textbf{A.} Slice through a cryo-tomogram of a FIB-milled horse sperm midpiece, showing mitochondria (mito), the submitochondrial reticulum (smr) outer dense fibers (odf), microtubule doublets (mtd), and the central pair apparatus (cpa).  Note how individual complexes (like the radial spoke, rs) are visible in the raw tomogram. The ordered protein array is only found on the axoneme-facing surface (yellow) of midpiece mitochondria, and not on the plasma membrane-facing surface (red). \textbf{B,C.} Slices through a cryo-tomogram of a FIB-milled horse sperm midpiece showing how the array directly interacts with the submitochondrial reticulum to anchor mitochondria to the flagellar  cytoskeleton (arrowheads). In right panels, the outer mitochondrial membrane is traced in green, the inner mitochondrial membrane in yellow, and the plasma membrane in blue. Scale bars: \textbf{A.} left - 250 nm, insets - 100 nm; \textbf{B,C.} 100 nm.}
	\label{fig:ch4_fig2}
\end{figure*}
Each boat-shaped particle rises $\sim$5 nm above the membrane and consists of two tilde-shaped densities arranged end-to-end. The boat-shaped structures form rows in which each particle is related to its closest neighbors by a $\sim$10 nm translation perpendicular to the particle long axis and a $\sim$6 nm shift along this axis, yielding a center-to-center spacing of $\sim$12 nm (\textbf{\autoref{fig:ch4_fig3}D-F}). Each row is oriented $\sim$120° to the long axis of the flagellum and adjacent rows are spaced $\sim$12 nm apart, forming extensive arrays on the axoneme-facing surface of the OMM (\textbf{\autoref{fig:ch4_fig3}G}). Remarkably, the averages we obtained from the three species were highly similar, both in terms of individual particle dimensions and in terms of their supramolecular arrangement (\textbf{\autoref{fig:ch4_fig3}}, \textbf{\autoref{fig:ch4_app_fig3}}). This conservation suggests that these arrays are a crucial structural element of the mitochondrial sheath.

Our averages revealed that the OMM underlying the protein arrays is studded with $\sim$3-4 nm pores arranged in a pseudo-lattice with a center-to-center spacing of $\sim$5 nm. (\textbf{\autoref{fig:ch4_fig3}A-C}, \textbf{\autoref{fig:ch4_app_fig3}G-I}). These pore sizes are consistent with the diameters of the voltage dependent anion channels (VDACs), which are known to form ordered arrays in the OMM \cite{Guo1993, Hoogenboom2007, Goncalves2007, Mannella1982}. VDACs are known to localize to the sperm midpiece \cite{Kwon2013, Arcelay2008}; indeed, our label-free quantitative proteomics experiments show that VDAC2 and VDAC3 are among the most abundant OMM proteins in pig sperm (\textbf{\autoref{tab:ch4_app_tab2}}). Furthermore, the lattice dimensions in our averages closely match those of VDAC in purified Neurospora OMM \cite{Guo1993, Mannella1998}. The lattice can be modeled by fitting multiple copies of the VDAC2 crystal structure \cite{Schredelseker2014} (\textbf{\autoref{fig:ch4_app_fig5}A}). We oriented VDAC2 in the membrane plane based on its known topology \cite{Tomasello2013, Bayrhuber2008}; however, at the current resolution, we cannot determine the orientation around the pore axis. Thus, in our model, each boat-shaped particle stretches across 8 VDAC molecules (\textbf{\autoref{fig:ch4_fig3}}).

\begin{figure*}[tb!]
	\center
	\includegraphics[]{Chapter.4/Figures/Figure3.png}
	\caption{\textbf{Ordered protein arrays at the mitochondria-cytoskeleton interface are conserved across species.} \textbf{A-C.} Subtomogram averages of the protein arrays and underlying outer mitochondrial membrane (OMM) after applying twofold symmetry (note that density is black). \textbf{D-F.} Isosurface renderings of the subtomogram averages in \textbf{A-C.} with boat-shaped particles in grey and the OMM in green. \textbf{G.} Left panel: Segmentation of the tomogram shown in Figure 2a, with the OMM in green, the IMM in yellow, microtubule doublets in blue, and the cpa in pink. Subtomogram averages of boat-shaped particles are colored grey and plotted back into their positions and orientations in the tomogram. Right panel: Rotated and zoomed-in view of the axoneme-facing surface of a mitochondrion. The axoneme is oriented horizontally, so the ladder-like arrays are oriented ~120° to the flagellar long axis, and individual particles within the array are oriented ~60° to this axis. Scale bars: \textbf{A-C.} 10 nm.}
	\label{fig:ch4_fig3}
\end{figure*}
%
\subsection*{Glycerol kinase-like proteins are probable constituents of the conserved arrays at the mitochondria-cytoskeleton interface}
To search for possible constituents of the protein arrays on the VDAC lattice, we used in-cell cross-linking mass spectrometry (XL-MS) \cite{Liu2018, Fasci2018} to find potential VDAC2 / VDAC3 interaction partners on the OMM (\textbf{\autoref{fig:ch4_fig4}}). We treated pig sperm cells with the cross-linker disuccinimidyl sulfoxide (DSSO), which covalently links free lysines that are within $\sim$3 nm (C$\alpha$-C$\alpha$) of each other. To increase confidence, we screened for protein interactions supported with at least two cross-link spectral matches (CSMs) (see Materials and Methods for details).

We first screened candidate proteins based on their known subcellular localizations (\textbf{\autoref{fig:ch4_fig4}A}). VDAC2/\linebreak VDAC3 cross-linked to mitochondria-associated proteins as well as to sperm head-associated proteins. This is consistent with immunofluorescence studies localizing VDAC2/VDAC3 both to the midpiece and to the acrosome, a large vesicle capping the anterior sperm nucleus \cite{Kwon2013, Hinsch2004, Arcelay2008}. Of the proteins in the mitochondria-associated interaction hub, four proteins are particularly noteworthy because they are known to localize to the OMM and because their disruption results in dysplasia of the mitochondrial sheath: armadillo repeat-containing protein 12 (ARMC12) \cite{Shimada2021}, spermatogenesis-associated protein 19 (SPATA19) \cite{Mi2015}, glutathione peroxidase 4 (GPX4) \cite{Schneider2009, Imai2009}, and glycerol kinase (GK) \cite{Chen2017a, Shimada2019}.

To distinguish among these candidates, we compared the location of the cross-links with the known topology of VDAC in the OMM \cite{Tomasello2013, Bayrhuber2008}. GPX4 would interact on the side facing the intermembrane space, whereas SPATA19, ARMC12, and GK would interact on the cytoplasmic face. Although we detect ARMC12 in mature pig sperm, in the mouse it is only present in developing spermatids and disappears from mature sperm \cite{Shimada2021}, which is inconsistent with the fact that the OMM arrays are present in cauda epididymal mouse sperm. SPATA19 and GK are both highly abundant (\textbf{\autoref{tab:ch4_app_tab2}}), as would be expected for proteins forming extensive arrays. Assuming an average protein density of $\sim$1.43 g/cm\textsuperscript{3} \cite{Quillin2000}, which corresponds to $\sim$0.861 Da/Å\textsuperscript{3}, we estimate that each boat-shaped particle in the array has a molecular weight of $\sim$250 kDa. SPATA19 is a small protein with an estimated molecular weight of $\sim$18 kDa. To fit into our EM densities, it must either be present in multiple copies or form a complex with other proteins. In contrast, GK has an estimated molecular weight of $\sim$60 kDa and is known to form S-shaped dimers ($\sim$120 kDa) that are conserved from bacteria \cite{Fukuda2016, Bystrom1999} to eukaryotes \cite{Balogun2019, Schnick2009}.

To build a GK-VDAC model based on our subtomogram average, we used rigid-body fitting to place two GK dimers end-to-end into a boat-shaped density (\textbf{\autoref{fig:ch4_app_fig5}B}, \textbf{\autoref{fig:ch4_fig4}B}). These fits defined a clear orientation for GK, with the N-termini pointing upwards and the C-terminal helices facing the OMM (\textbf{\autoref{fig:ch4_fig4}B}). To validate our fits, we mapped the cross-linked lysines onto the resulting model (\textbf{\autoref{fig:ch4_fig4}C}). All cross-links were between the cytosolic face of VDAC2 and the OMM-facing surface of GK, which is consistent with the orientation expected from our fits. These fits show that the molecular dimensions and relative positions of candidate proteins are consistent with our subtomogram average maps, but we cannot define specific interaction sites at the current resolution. We also attempted to analyze interactions between the putative GK-like proteins and the underlying cytoskeleton, and although we detect cross-links on the putative cytoskeleton-facing side of GK (\textbf{\autoref{fig:ch4_app_fig6}}), interpretation is complicated by the fact that the precise protein compositions of the submitochondrial reticulum and the ODFs are unclear.

Assigning GK-like proteins as constituents of the ordered OMM arrays at the mito\-chondria-cytoskeleton interface is also supported by recent genetic studies. Sperm from mice lacking sperm-specific GK isoforms have disorganized mitochondrial sheaths \cite{Chen2017a, Shimada2019}. In these mice, spherical mitochondria properly align along the flagellum but fail to properly elongate and coil around the flagellum, leaving gaps along the midpiece where the ODFs are exposed \cite{Shimada2019}. This phenotype is consistent with our data showing contacts between GK protein arrays and the underlying cytoskeleton (\textbf{\autoref{fig:ch4_fig2}B-C}, \textbf{\autoref{fig:ch4_app_fig4}S-L}), and by experiments showing that the submitochondrial reticulum remains attached to the OMM in detergent-extracted sperm (\cite{Olson1990}. Although sperm-specific GK isoforms do not show glycerol kinase activity in vitro \cite{Pan1999}, we cannot completely exclude that they function in mitochondrial metabolism \emph{in situ}, especially since knockout mice have higher ATP levels than wildtype (Chen2017a). If the arrays we observe indeed consist of GK-like proteins, and if these proteins are in fact metabolically active, then their preferential orientation towards the flagellar cytoskeleton could enhance metabolite shuttling between mitochondria and the axoneme.
\begin{figure*}[p]
	\center
	\includegraphics[]{Chapter.4/Figures/Figure4.png}
	\caption{Figure Legend on next page}
	\label{fig:ch4_fig4}
\end{figure*}
\addtocounter{figure}{-1}
\begin{figure*}[ht]
	\caption{\textbf{Modelling the outer mitochondrial membrane (OMM) array as glycerol kinase-like (GK) proteins anchored on voltage dependent anion channels (VDACs).} \textbf{A.} The VDAC2/VDAC3 interactome derived from in-cell XL-MS of pig sperm. Protein nodes are colored according to their known subcellular localizations (yellow with red border - experimentally verified to localize to sperm mitochondria; yellow - gene ontology: mitochondria, blue - gene ontology: nucleus/acrosome/cell surface; grey - gene ontology: cytoskeleton/unknown). Gray spheres indicate the phosphate groups of a simulated lipid bilayer which was structurally aligned based on the simulation for  monomeric mouse VDAC1 (PDB 4C69) obtained from the MemProtMD server \cite{Newport2019}. \textbf{B.} Modeling the OMM array as GK-like proteins anchored on VDACs. A GK-like dimer-of-dimers homology model (red and green) and VDAC2 homology models (yellow) were fitted into the pig subtomogram average map (white). \textbf{C.} The positions of cross-linked Lys residues (red circles) are consistent with GK and VDAC orientation assignments in our model. Note that the cross-links are shown for VDAC2, but we detect an additional cross-link between K140 on GK and VDAC3 (see \textbf{\autoref{fig:ch4_app_fig6}}).}
\end{figure*}
%
\section{Discussion}
In this study, we used cryo-FIB milling-enabled cryo-ET to image the sperm mitochondrial sheath in three mammalian species. Our data reveal that overall mitochondrial dimensions are remarkably consistent in sperm from the same species (\textbf{\autoref{fig:ch4_fig1}}, \textbf{\autoref{fig:ch4_app_fig1}}). This contrasts with other mitochondria-rich tissues such as muscle, where there are massive variations in mitochondrial size and morphology within individual cells \cite{Vincent2019}. In addition, we did not observe mitochondrial nanotunnels in any of the species we examined, in contrast to their relative abundance in muscle tissue \cite{Vincent2017, Vincent2019}. However, we do observe discrete linker proteins that tether neighboring mitochondria in all three species. It will be important to investigate the precise molecular identity of these linkers and to define whether they serve purely structural roles or also act as conducting channels that coordinate metabolite shuttling, energy production, and calcium signaling across the entire midpiece. Hinting at possible inter-organelle coordination, we observed trans-mitochondrial cristae alignment at specialized junctions between neighboring mitochondria in mouse sperm (\textbf{\autoref{fig:ch4_fig1}}). Thus, the concept of an interconnected mitochondrial reticulum that occurs in muscle \cite{Glancy2015} may also be applicable to the ma\-mmalian sperm midpiece and warrants further consideration.

In this study, we imaged either ejaculated or mature cauda epididymal sperm. However, mammalian sperm must undergo a plethora of biochemical and morphological changes in the female reproductive tract before they become fertilization-competent. These processes are collectively known as capacitation and cause extensive metabolic changes in sperm \cite{Mounib1964}. To cope with increased energy demands, rates of both glycolysis and oxidative phosphorylation increase during capacitation in mouse sperm \cite{Balbach2020}. Future work could use cryo-ET to establish whether these metabolic changes relate to any structural changes in midpiece mitochondria, for instance at the level of cristae architecture or intermitochondrial junctions.

Our data also show that mitochondrial dimensions and cristae architecture vary across species (\textbf{\autoref{fig:ch4_fig1}}), providing possible structural bases for interspecific differences in mitochondrial energetics. Horse sperm mitochondria appear to take on a more condensed appearance than their counterparts in pig and mouse, which may correlate with increased reliance on oxidative phosphorylation in horse sperm. Although studies on horse sperm physiology are limited \cite{Meyers2019}, condensed mitochondrial profiles are suggested to be a sign of calcium overload in other species \cite{Jamil1981, Okunade2004}, so cross-species studies exploring the link between mitochondrial morphology and possible differences in mitochondrial calcium homeostasis are clearly necessary.In general, the natural diversity of sperm enables comparative studies of how mitochondrial structure varies with sperm metabolism, which will undoubtedly contribute to our broader understanding of how mitochondrial form relates to function.

Our data show that, despite this diversity, the molecular underpinnings of mitochondrial sheath architecture are conserved, at least in mammals. Specifically, we identified novel intermitochondrial linkers that tether adjacent mitochondria (\textbf{\autoref{fig:ch4_fig1}}, \textbf{\autoref{fig:ch4_app_fig2}}). We also describe ordered protein arrays on the axoneme-facing surface of sperm mitochondria, which were first identified by freeze-fracture EM. Here, we characterize the three-dimensional architecture of these arrays in a near-native state and at molecular resolution, revealing how they are anchored on the OMM and how they interact with the flagellar  cytoskeleton. In-cell subtomogram averaging reveals that these arrays consist of boat-shaped particles anchored on a lattice of OMM pores (\textbf{\autoref{fig:ch4_fig2}}, \textbf{\autoref{fig:ch4_fig3}}). Proteomics and in-cell XL-MS suggest that these arrays consist of GK-like proteins anchored on VDAC lattices in the OMM (\textbf{\autoref{fig:ch4_fig4}}). Given that VDACs are ubiquitous OMM proteins, our findings motivate further efforts to explore whether they also regulate mitochondria-cytoskeleton interactions in other cell types.

The OMM arrays may function to regulate the precise elongation and coiling of mitochondria, contributing to the striking consistency within the mitochondrial sheath. In mature sperm, these arrays may help maintain the integrity of mitochondria-cytoskeleton contacts, stabilizing them against shear stresses during sperm motility and hyperactivation. However, it is unclear what determines the organization of these arrays in the first place. Our averages do not hint at direct interactions between boat-shaped particles. Instead, their spacing may be defined by the organization of the underlying VDAC lattice. Another intriguing possibility is that the arrays are organized by their cytoskeletal binding partners; the periodicity of relevant motifs on the submitochondrial reticulum could dictate the spacing of the OMM arrays.

We find that mitochondria-cytoskeleton contact sites in the sperm midpiece can be quite variable \linebreak (\textbf{\autoref{fig:ch4_app_fig4}}). Occasionally, OMM arrays make direct contact with the ODFs (\textbf{\autoref{fig:ch4_app_fig4}A-C,E-F,I}); sometimes, there are considerable gaps between them that are bridged either by the submitochondrial reticulum (\textbf{\autoref{fig:ch4_app_fig4}D,G-H,J}) or by unidentified linkers (\textbf{\autoref{fig:ch4_app_fig4}K-L}). This may be explained by variability in both the sizes and shapes of the ODFs themselves (\textbf{\autoref{fig:ch4_app_fig4}M-R}). Defining the precise mechanisms by which the OMM arrays tether to the underlying cytoskeleton is an important avenue for future research, in part because it has important implications for understanding sperm motility. In mammalian sperm, forces from axoneme bending are transmitted to the connecting piece through the ODFs, which are physically coupled to the microtubule doublets along most of the principal piece but not in the midpiece \cite{Lesich2014, Leung2021}. The observation that mitochondria are anchored to the ODFs through extensive ordered arrays adds another level of complexity to the multi-layered sliding mechanics of the mammalian sperm flagellum, affecting for example the extent to which the ODFs can slide relative to each other or to the axoneme. Indeed, the infertility of mice lacking sperm-specific GK isoforms appears to be caused by motility defects arising from the fragmented mitochondrial sheath \cite{Chen2017a, Shimada2019}. intermitochondrial linkers that physically tether neighboring mitochondria may also affect the extent to which the midpiece can bend during movement. On a larger scale, inter-species variations in mitochondrial dimensions and packing, coupled with known differences in overall midpiece length, could result in subtle changes in sperm motility. Furthermore, the stiffness of the midpiece changes during sperm maturation, becoming less rigid as sperm transit through the epididymis \cite{Jeulin1996, Miyata2015}. Tracing how the molecular structures we describe here form and change during spermiogenesis and epididymal maturation are likely to provide further insight in this respect. Ultimately, combining structural information from cryo-ET with motility analysis and mathematical modelling could help illuminate how midpiece morphometry affects swimming behaviour.

The methods we use here can also be used to study other specialized organelle configurations in the highly-streamlined sperm cell. For example, it is unclear how exactly the sperm head anchors to the tail, or how the acrosomal vesicle is laminated to the nucleus. Beyond sperm, such methods could be used to gain molecular insight into the organization of organelles in other terminally-differentiated and polarized cell types like neurons, photoreceptors, or hair cells. For example, exploring whether similar arrays are present at the mitochondria-cytoskeleton interface in other differentiated cell types - and whether they use a similar pool of protein components - is an area ripe for study. In striated muscle, proper mitochondrial positioning is critical for muscle function and depends on direct associations between mitochondria and intermediate filaments \cite{Konieczny2008, Milner2000}. Similarly, in skin cells, mitochondrial organization depends on keratin \cite{Steen2020}. The structural bases for these associations are unknown, but cryo-ET and in-cell XL-MS may prove useful in these contexts as well.
%%Material and Methods
\section{Materials and Methods}
\subsection*{Sperm collection and preparation}
Pig sperm samples were obtained from Varkens KI Nederland and prepared for imaging within a few hours of delivery. Sperm were layered onto a discontinuous gradient consisting of 4 mL of 35\% Percoll\textsuperscript{®} (GE Healthcare) underlaid with 2 mL of 70\% Percoll\textsuperscript{®}, both in HEPES-buffered saline (HBS: 20 mM HEPES, 137 mM NaCl, 10 mM glucose, 2.5 mM KCl, 0.1\% kanamycin, pH 7.6) and centrifuged at 750g for 15 min at room temperature (RT). Pelleted cells were washed once in phosphate-buffered saline (PBS: 137 mM NaCl, 3 mM KCl, 8 mM Na2HPO4, 1.5 mM KH2PO4, pH 7.4), and resuspended in PBS for further processing. All solutions were iso-osmotic at RT (290-300 mOsm/kg).

Horse semen was collected from mature Warmblood stallions using a Hanover artificial vagina in the presence of a teaser mare. Semen was filtered through gauze and kept at room temperature until further processing. Semen was diluted in INRA96 (IMV Technologies) to a sperm concentration of 30 x 106 cells/mL. Sperm were then centrifuged through a Percoll\textsuperscript{®} gradient (as described above for the pig) for 10 min at 300g followed by 10 min at 750g \cite{Harrison1993}. The pellet was resuspended in 1 mL of PBS, centrifuged for 5 min at 750g, and finally resuspended in PBS.

Mouse sperm were obtained from the cauda epididymis of adult male C57BL/6 mice as described in \cite{Hutcheon2017}. Male mice were culled as described in \cite{Mederacke2015} and the cauda epididymides were dissected with the vas deferens attached. Tissues were placed in a 500 µL droplet of modified Biggers, Whitten, and Whittingham media (BWW: 20 mM HEPES, 91.5 mM NaCl, 4.6 mM KCl, 1.7 mM D-glucose, 0.27 mM sodium pyruvate, 44 mM sodium lactate, 5 U/mL penicillin, and 5 µg/mL streptomycin, adjusted to pH 7.4 and an osmolarity of 300 mOsm/kg). Sperm were gently pushed out from the vas deferens, after which two incisions were made with a razor blade in the cauda. Spermatozoa were allowed to swim out of the cauda into the BWW over a period of 15 min at 37°C, after which the tissue was removed and sperm were loaded onto a 27\% Percoll\textsuperscript{®} density gradient and washed by centrifugation at 400g for 15 min. The sperm pellet was resuspended in BWW and centrifuged at 400g for 2 min to remove excess Percoll\textsuperscript{®}, then finally resuspended in BWW.
%
\subsection*{Cryo-EM grid preparation}
Typically, 3 µL of sperm suspension was pipetted onto a glow-discharged Quantifoil R 2/1 200-mesh holey carbon grid. Sperm cell density was approximately 2-3 x 106 cells/mL (for whole cell tomography) or 20-30 x 106 cells/mL (for cryo-FIB milling). One µL of BSA-conjugated gold (Aurion) was added, and the grids blotted manually opposite the side of cell deposition for $\sim$3 s (for whole cell tomography) or for $\sim$5-6 s (for cryo-FIB milling) using a manual plunge-freezer (MPI Martinsreid). Grids were immediately plunged into a liquid ethane-propane mix (37\% ethane) \cite{Tivol2008} cooled to liquid nitrogen temperature. Grids were stored under liquid N2 until imaging.
%
\subsection*{Cryo-focused ion beam milling}
Grids were mounted into modified Autogrids (ThermoFisher) for mechanical support. Clipped grids were loaded into an Aquilos (ThermoFisher) dual-beam cryo-focused ion beam/scanning electron microscope (cryo-FIB/SEM). SEM imaging was performed at 2 kV and 13 pA, whereas FIB imaging for targeting was performed at 30 kV and 10 pA. Milling was performed with stage tilts between 15° and 18°, so lamellae were inclined 8-11° relative to the grid.  Each lamella was milled in four stages: an initial rough mill at 1 nA beam current, an intermediate mill at 300 pA, a fine mill at 100 pA, and a polishing step at 30 pA. Lamellae were milled with the wedge pre-milling technique described in \cite{Schaffer2017} and with expansion segments as described in \cite{Wolff2019}.
%
\subsection*{Tilt series acquisition}
Tilt series were acquired on either a Talos Arctica (ThermoFisher) operating at 200 kV (all lamellae datasets and the whole-cell VPP horse dataset) or a Titan Krios (ThermoFisher) operating at 300 kV (whole-cell VPP pig and mouse datasets). Both microscopes were equipped with a post-column energy filter (Gatan) operated in zero-loss imaging mode with a 20-eV energy-selecting slit. All images were recorded on a K2 Summit direct electron detector (Gatan) in either counting or super-resolution mode with dose-fractionation. Tilt series were collected using SerialEM \cite{Mastronarde2005} at target defocus values between -4 and -6 $\mu$m (conventional defocus-contrast) or between -0.5 and -1.5 $\mu$m (for tilt series acquired with the Volta phase plate). Tilt series were typically recorded using either strict or grouped dose-symmetric schemes, either spanning ± 56° in 2° increments or ± 54° in 3° increments, with total dose limited to $\sim$100 e-/Å\textsuperscript{2}.
%
\subsection*{Tomogram reconstruction}
Frames were aligned either post-acquisition using Motioncor2 1.2.1 \cite{Zheng2017} or on-the-fly using Warp \cite{Tegunov2019}. Frames were usually collected in counting mode; when super-resolution frames were used (whole-cell VPP pig dataset), they were binned 2X during motion correction. Tomograms were reconstructed in IMOD \cite{Kremer1996} using weighted back-projection, with a SIRT-like filter \cite{Zeng2012} to aid visualization and segmentation. Defocus-contrast tomograms were CTF-corrected in IMOD using ctfphaseflip. VPP tomograms were left uncorrected.
%
\subsection*{Tomogram segmentation}
Segmentation was performed semi-automatically using the neural network-based workflow implemented in the TomoSeg package in EMAN 2.21 \cite{Chen2017b} Microtubules were traced manually in IMOD. Segmentation was then manually refined in Chimera 1.12 \cite{Pettersen2004} or in ChimeraX \cite{Goddard2018}. Visualization was performed in ChimeraX.
%
\subsection*{Subtomogram averaging of ATP synthase and ladder-like arrays}
Subtomogram averaging with missing wedge compensation was performed using PEET 1.13.0 \cite{Heumann2011, Nicastro2006} on defocus-contrast tomograms of cryo-FIB-milled lamellae. Resolution was estimated using the Fourier shell correlation (FSC) at a cut-off of 0.5 \cite{Nicastro2006}. Alignments were generally performed first on binned data, after which aligned positions and orientations were transformed to less-binned data using scripts provided by Dr. Daven Vasishtan. Details of acquisition parameters and particle numbers are summarized in \textbf{\autoref{tab:ch4_app_tab1}}.

Alignment strategies for mitochondrial complexes were designed to take advantage of their defined orientations relative to the membrane plane. Particles were picked manually and their initial orientations were defined using \emph{stalkInit}. Initial references were either a randomly chosen particle (for ladder-like arrays) or an average of all particles after roughly aligning them based on their initial orientations (for ATP synthase). Independent alignments using independent initial references were performed for datasets from different species. Alignments allowed for large rotational search ranges around the particle long axis (defined as the y-axis, perpendicular to the membrane plane), with limited search ranges around the x- and z-axes (the membrane plane).

All initial alignments were performed without symmetry. After visual inspection of the maps, twofold symmetry was applied for ladder-like arrays. Symmetrization involved using the aligned positions from the unsymmetrized runs as seed points and rotating particles around the axis of symmetry to generate virtual particles. A symmetrized volume was generated by averaging all particles and virtual particles and used as a reference for a final, restricted alignment.

Plotbacks were generated in IMOD by first running createAlignedModel to generate model files reflecting updated particle positions and orientations after alignment. The relevant subtomogram average was then thresholded for visualization and saved as an isosurface model, which was then placed back into the tomograms using \emph{clonemodel}.
%
\subsection*{Measurements and quantification}
All measurements of mitochondrial width were performed in IMOD on Volta phase plate tomograms filtered with a SIRT-like filter. Mitochondrial width was measured in the non-missing wedge direction at five points along the length of each mitochondrion. Only mitochondria that were entirely in the field of view were included in the measurements. Tomograms and corresponding measurements were then grouped based on their locations relative to the connecting piece, which were determined based on low-magnification images used for targeting during data acquisition. Internal mitochondrial ultrastructure was quantified from tomograms from cryo-FIB milled lamellae. The volume occupied by the matrix (V\textsubscript{\emph{matrix}}, the volume enclosed by the IMM) was measured relative to the volume occupied by the entire mitochondrion (V\textsubscript{\emph{mito}}, the volume enclosed by the OMM). Mesh volumes were extracted from segmentations using \emph{imodinfo}. Because neural network-based segmentation often resulted in gaps, mitochondrial membranes were segmented manually in IMOD for quantification. Only slices in which both the IMM and OMM were clearly defined were used for segmentation.
%
\subsection*{Cross-linking, lysis, digestion, and peptide fractionation}
All proteomics and cross-linking mass spectrometry experiments were performed on Percoll-washed pig sperm prepared as described above. For cross-linking, approximately 300 x 10\textsuperscript{6} cells were used, each from 3 different animals. Briefly, pelleted sperm cells were resuspended in 540 $\mu$L of PBS supplemented with disuccinimidyl sulfoxide (DSSO, Thermo Fisher Scientific) to a final concentration of 1 mM. The reaction mix was incubated for 30 min at 25°C with 700 rpm shaking in a ThemoMixer C (Eppendorf) and subsequently quenched for 20 min by adding Tris-HCl (final concentration 50 mM). Cross-linked cells were spun down at 13800g for 10 min at 4°C, after which the supernatant was removed. Cells were then lysed according to a protocol modified from \cite{Potel2018}. Cells were resuspended in 1 mL of lysis buffer (100 mM Tris-HCl pH 8.5, 7 M Urea, 1\% Triton X-100, 5 mM TCEP, 30 mM CAA, 10 U/ml DNase I, 1 mM MgCl\textsubscript{2}, 1\% benzonase (Merck Millipore, Darmstadt, Germany), 1 mM sodium orthovanadate, phosphoSTOP phosphatases inhibitors, and cOmpleteTM Mini EDTA-free protease inhibitors). Cells were sonicated on ice for 2 min using an ultrasonic processor (UP100H, Hielscher) at 80\% amplitude. The proteins were then precipitated according to \cite{Wessel1984} and the dried protein pellet resuspended in digestion buffer (100 mM Tris-HCl pH 8.5, 1\% sodium deoxycholate (Sigma-Aldrich), 5 mM TCEP, and 30 mM CAA). Trypsin and Lys-C proteases were added to a 1:25 and 1:100 ratio (w/w) respectively and protein digestion performed overnight at 37°C. The final peptide mixtures were desalted with solid-phase extraction C18 columns (Sep-Pak, Waters) and fractionated with an Agilent 1200 HPLC pump system (Agilent) coupled to a strong cation exchange (SCX) separation column (Luna SCX 5 $\mu$m - 100 Å particles, 50 x 2mm, Phenomenex), resulting in 25 fractions.
%
\subsection*{Liquid chromatography with mass spectrometry}
Approximately 1000 ng of peptides from each biological replicate before SCX fractionation were first injected onto an Agilent 1290 Infinity UHPLC system (Agilent) on a 50-cm analytical column packed with C18 beads (Dr Maisch Reprosil C18, 3 µm) coupled online to an Orbitrap HF-X (Thermo Fisher Scientific). For this classical bottom-up analysis, we used the following LC-MS/MS parameters: after 5 min of loading with 100\% buffer A (H2O with 0.1\% formic acid), peptides were eluted at 300 nL/min with a 95-min gradient from 13\% to 40\% of buffer B (80\% acetonitrile and 20\% H2O with 0.1\% formic acid). For MS acquisition we used an MS1 Orbitrap scan at 60,000 resolution, automatic gain control (AGC) target of 3 x 10\textsuperscript{6} ions and maximum inject time of 20 ms from 375 to 1600 m/z; the 15 most intense ions were submitted to MS2 Orbitrap scan at 30,000 resolution, AGC target of 1 x 10\textsuperscript{5} ions and maximum inject time of 50 ms (isolation window of 1.4 m/z, NCE at 27\% and dynamic exclusion of 16 seconds). The SCX fractions were analyzed with same Agilent HPLC and the same nano-column coupled on-line to an Orbitrap Lumos mass spectrometer (ThermoFisher Scientific). For these runs, we used a gradient from 6\% to 39\% buffer B over 100 min with specific MS settings for DSSO cross-links: survey MS1 Orbitrap scan at 60,000 resolution from 375 to 1500, AGC target of 4 x 10\textsuperscript{5} ions and maximum inject time of 50 ms; MS2 Orbitrap scan at 30,000 resolution, AGC target of 5 x 10\textsuperscript{4} ions, and maximum inject time of 100 ms for detection of DSSO signature peaks (difference in mass of 37.972 Da). The four ions with this specific difference were analyzed with a MS3 Ion Trap scans at AGC target of 2 x 10\textsuperscript{4} ions, maximum inject time of 150 ms for sequencing selected signature peaks (representing the individual peptides).
%
\subsection*{Data processing}
The 3 raw files obtained with classical bottom-up approach were analyzed with Max\-Quant version 1.6.17 with all the automatic settings adding Deamidation (N) as dynamic modification against the Sus scrofa reference proteome (Uniprot version of 08/2020 with 49,795 entries). With this search, we were able to calculate intensity-based absolute quantification (iBAQ) values and created a smaller FASTA file to use for analysis of cross-linking experiments. Raw files for cross-linked cells were analyzed with Proteome Discoverer software suite version 2.4.1.15 (ThermoFisher Scientific) with the incorporated XlinkX node for analysis of cross-linked peptides as described in \cite{Klykov2018}. Data were searched against the smaller FASTA created in house with “MS2\_MS3 acquisition strategy”. For the XlinkX search, we selected full tryptic digestion with 3 maximum missed cleavages, 10 ppm error for MS1, 20 ppm for MS2, and 0.5 Da for MS3 in Ion Trap. For modifications, we used static Carbamidomethyl (C) and dynamic Oxidation (M), Deamidation (N) and Met-loss (protein N-term). The crosslinked peptides were accepted with a minimum score of 40, minimum score difference of 4 and maximum FDR rate set to 5\%; further standard settings were used.
%
\subsection*{Interactome analysis, homology modelling, and cross-link mapping}
The interaction map for VDAC proteins was generated in R \cite{Grant2006} using the igraph package (v 1.2.4.2). Only protein interactions supported with at least two cross-link spectral matches (CSMs) were included in the final network. Homology models of pig GK and pig VDAC2 were generated in Robetta \cite{Kim2004} and fitted into subtomogram average maps by rigid body fitting in ChimeraX. Cross-links were mapped onto the resulting models using ChimeraX.
%
\subsection*{Data availability}
Subtomogram average maps have been deposited to the Electron Microscopy Data Bank (EMDB) with the following accession numbers: EMDB-12354, 12355, 12356, and 12357. The model of putative glycerol kinase-like proteins anchored on a VDAC array has been deposited to the Protein Data Bank (PDB) with the accession number PDB ID 7NIE. Mass spectrometry data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD025562.
%
\subsection*{Acknowledgments}
The authors thank Dr. M Vanevic for excellent computational support; Dr. D Vasishtan for sharing scripts for subtomogram averaging; Ingr. CTWM Schneijdenberg and JD Meeldijk for managing the Utrecht University EM Square facility; Stal Schep (Tull en het Waal, The Netherlands) for providing horse semen; DVM MW Haaker and Dr. M Houweling for providing mouse reproductive tracts; Prof. F Förster and Prof. A Akhmanova for their comments on early versions of the manuscript; and Prof. EY Jones for insightful discussions. This work benefitted from access to the Netherlands Center for Electron Nanoscopy (NeCEN) with support from operators Dr. RS Dillard and Dr. C Diebolder and IT support from B Alewijnse. RZC, JFH, and AJRH acknowledge support from NWO funding the Netherlands Proteomics Centre through the X-omics Road Map program (project 184.034.019). This work was funded by NWO Start-Up Grant 740.018.007 to TZ. MRL is supported by a Clarendon Fund-Nuffield Department of Medicine Prize Studentship..
%
\subsection*{Author contributions}
PM, MZ, HH, EGB, and BMG provided sperm samples. MRL, MCR, and RTR prepared samples for cryo-ET. MRL performed cryo-FIB milling. MRL, MCR, RTR, SCH, and TZ collected cryo-ET data. MRL and MCR processed cryo-ET data. MRL, MCR, and TZ analyzed cryo-ET data. RZC and JFH performed all proteomics and cross-linking mass spectrometry experiments and corresponding structural modelling under the supervision of AJRH. MRL and TZ wrote the manuscript, and all authors contributed to revisions.
%
\subsection*{Conflict of interest}
The authors declare no competing interests.
\clearpage
%%Appendix
\begin{subappendices}
	%%Renew table numbering format
	\makeatletter\renewcommand{\thetable}{\thechapter.A.\@arabic\c@table}
	\makeatother
	\counterwithin{figure}{section}
	\section{Supplementary Material}
	\begin{figure*}[hbt]
		\center
		\includegraphics[]{Chapter.4/Figures/SI_Figure1.png}
		\caption{\textbf{Mitochondrial dimensions are consistent within species but vary across species and spatially along the midpiece.} \textbf{A.} Plotting the average width of mitochondria from different regions of the midpiece shows that mouse sperm mitochondria are larger than pig and horse sperm mitochondria. Note also that, in all three species, mitochondria at the proximal end of the midpiece are larger than those in more distal parts. \textbf{B.} Mitochondrial width was measured at five points along the length of each mitochondrion. Plotting the coefficient of variation from different regions along the midpiece shows that mitochondria at the start of the midpiece have more variable widths along their lengths. In \textbf{A.}, n indicates the number of mitochondria analyzed. In both \textbf{A.} and \textbf{B.}, solid lines represent the median and dotted lines represent the first and third quartiles.}
		\label{fig:ch4_app_fig1}
	\end{figure*}
	\begin{figure*}[hbt]
		\center
		\includegraphics[]{Chapter.4/Figures/SI_Figure2.png}
		\caption{\textbf{Cryo-focused ion beam (cryo-FIB) milling reveals the internal organization of sperm mitochondria.} \textbf{A.} Slice through a cryo-tomogram of FIB-milled pig sperm mitochondria close to the connecting piece. \textbf{B.} ATP synthase can be directly identified on cristae based on its characteristic shape, which is confirmed by subtomogram averaging \textbf{B'}. \textbf{C.} Novel inter-mitochondrial linkers tether neighboring mitochondria to each other (arrowheads in left panel). \textbf{D.} Quantifying the volume enclosed by the mitochondrial matrix relative to the volume enclosed by the whole mitochondrion reveals that pig and horse sperm mitochondria have more expanded cristae and more condensed matrices than mouse sperm mitochondria. Lines represent mean ± standard deviation. Scale bars: \textbf{A.} 250 nm, \textbf{B-C.} 100 nm.}
		\label{fig:ch4_app_fig2}
	\end{figure*}
	\begin{figure*}[hbt]
		\center
		\includegraphics[]{Chapter.4/Figures/SI_Figure3.png}
		\caption{\textbf{The particles forming the ordered arrays at the mitochondria-cytoskeleton interface are two-fold symmetric.} \textbf{A-F.} Slices through cryo-tomograms of FIB-milled pig \textbf{A-B.}, horse \textbf{C-D.}, and mouse \textbf{E-F.} mitochondria. Right panels show digital zooms of the regions boxed out in the left panels, with arrowheads indicating arrays. \textbf{G-I.} Subtomogram averages of the arrays and the outer mitochondrial membrane (OMM) without (left) and with (right) twofold symmetry. Scale bars: \textbf{A-F.} 250 nm, \textbf{G-I.} 10 nm.}
		\label{fig:ch4_app_fig3}
	\end{figure*}
	\begin{figure*}[hbt]
		\center
		\includegraphics[]{Chapter.4/Figures/SI_Figure4.png}
		\caption{\textbf{Morphology of mitochondria-cytoskeleton contacts in the mammalian sperm midpiece.} \textbf{A-L} Slices through cryo-tomograms of FIB-milled pig (left), horse (middle), and mouse (right) sperm illustrating the variable nature of mitochondria-cytoskeleton contacts. Occasionally, the OMM array makes contact with the ODFs \textbf{A-C}, \textbf{E-F}, \textbf{I}; in other cases, the space between them is bridged by the submitochondrial reticulum \textbf{D,G-H,J} or by unidentified linkers \textbf{K-L}. \textbf{M-R} Transverse slices through cryo-tomograms illustrating how variability in size and shape of the outer dense fibers (ODFs) leads to variations in mitochondria-ODF spacing. Labels: odf - outer dense fiber, mito - mitochondria. Scale bars: \textbf{A-L} 50 nm, \textbf{M-R} 100 nm.}
		\label{fig:ch4_app_fig4}
	\end{figure*}
	\begin{figure*}[hbt]
		\center
		\includegraphics[]{Chapter.4/Figures/SI_Figure5.png}
		\caption{\textbf{Fitting crystal structures of glycerol kinase (GK) and voltage dependent anion channels (VDACs) into the pig subtomogram average map.} \textbf{A} The crystal structure of VDAC2 from zebrafish (PDB 4BUM) is shown in grey, fitted into the cryo-ET averaged map (green). \textbf{B} Two copies of a crystal structure of GK (pink and blue) from Trypanosoma brucei (PDB 5GN6) fitted into the cryo-ET averaged map (grey). On the right, the GK crystal structure is shown filtered to 30Å resolution.}
		\label{fig:ch4_app_fig5}
	\end{figure*}
	\begin{figure*}[hbt]
		\center
		\includegraphics[]{Chapter.4/Figures/SI_Figure6.png}
		\caption{\textbf{The interactome of glycerol kinase (GK), a putative constituent of the boat-shaped particles on the ordered outer mitochondrial membrane array.} Each direct interaction partner of GK is connected by a dotted line to the lysine residue with which it cross-linked. Cross-linked lysine residues on GK are mapped onto a homology model of the pig GK dimer, which in turn is oriented based on our subtomogram average maps.}
		\label{fig:ch4_app_fig6}
	\end{figure*}
	\begin{table*}[hbt!]
		\caption{\textbf{Image acquisition and processing metrics for subtomogram averaging of mitochondrial protein complexes in mammalian
				sperm.}}
		\includegraphics[]{Chapter.4/Figures/SI_Table1.png}
		\label{tab:ch4_app_tab1}
	\end{table*}
	\begin{table*}[hbt!]
		\caption{\textbf{Top 35 most abundant outer mitochondrial membrane proteins identified in the pig sperm proteome.}}
		\includegraphics[]{Chapter.4/Figures/SI_Table2.png}
		\label{tab:ch4_app_tab2}
	\end{table*}
\end{subappendices}
\clearpage
%%References
\section*{References}
\bibliographystyle{Style_settings/bibstyle_pnas}
\bibliography{Chapter.4/chapter4_bib}
