\picturechapter{Molecular characterization of a complex of apoptosis-inducing factor 1 with cytochrome \emph{c} oxidase of the mitochondrial respiratory chain}{Chapter_covers/chapter_cover_3.pdf} \label{ch-3}
\vspace*{0.25cm}

{\footnotesize Johannes F. Hevler, Riccardo Zenezeni Chiozzi, Alfredo Cabrera-Orefic, Ulrich Brandt, Susanne Arnold and Albert J.R. Heck}
%
\begin{center}
    \vspace{3cm}
    \includegraphics[]{Chapter.3/Figures/COX_AIFM1_Goodshell_like.png}
    \vspace{0.25cm}
\end{center}
%
\begin{flushleft}
    \vspace*{\fill}
    \rule{\textwidth}{1pt}\\[0cm]
    \textbf{This chapter is based on work in the following publication:}\\
    \footnotesize
    \textbf{\emph{PNAS}} (2021), 118:e2106950118, doi:10.1073/pnas.2106950118\\
    \footnotesize
\end{flushleft}
%%Abstract
\begin{abstract102}
    Combining mass spectrometry based chemical cross-linking and complexome profiling, we analyzed the interactome of heart mitochondria. We focused on complexes of oxidative phosphorylation and found that dimeric apoptosis inducing factor 1 (AIFM1) forms a defined complex with $\sim$10\% of monomeric cytochrome \emph{c} oxidase (COX), but hardly interacts with respiratory chain supercomplexes. Multiple AIFM1 inter-cross-links engaging six different COX subunits provided structural restraints to build a detailed atomic model of the COX-AIFM1\textsubscript{2} complex (PDBDEV\_00000092). Application of two complementary proteomic approaches thus provided unexpected insight into the macromolecular organization of the mitochondrial complexome. Our structural model excludes direct electron transfer between AIFM1 and COX. Notably however, the binding site of cytochrome \emph{c} remains accessible allowing formation of a ternary complex. The discovery of the previously overlooked COX-AIFM1\textsubscript{2} complex and clues provided by the structural model hint at potential roles of AIFM1 in OXPHOS biogenesis and in programmed cell death.
\end{abstract102}
%%Main Text
\thumbforchapter
\section{Introduction}
\lettrine[lraise=0.1, nindent=0em, slope=-.5em]{M}{itochondria} are considered the powerhouse of aerobic eukaryotic cells, as they contain the major pathways of oxidative energy metabolism and produce the bulk of ATP by oxidative phosphorylation (OXPHOS) necessary for cellular homeostasis. Only at the end of the last century it became evident that mitochondria also are key players in apoptosis and that this process is tightly linked to OXPHOS components \cite{RN1}. Apoptosis Inducing Factor (AIFM1) was one of the proteins found to be released from the mitochondrial intermembrane space during programmed cell death and to have the capacity to induce chromatin condensation and DNA fragmentation in a caspase-independent fashion \cite{RN2}. A mutation found in AIFM1 has been associated with Cowchock syndrome (OMIM 310490) \cite{RN3}. Early on, it was also reported that ablation of AIFM1 leads to OXPHOS deficiency \cite{RN4}, in line with findings that AIFM1 mutations cause combined oxidative phosphorylation deficiency 6, a severe mitochondrial encephalomyopathy (OMIM 300816; \cite{RN5}). More recently, it has been proposed that AIFM1 is involved in the disulfide relay of the mitochondrial intermembrane space by serving as import receptor of CHCHD4/MIA40 \cite{RN8, RN7, RN6}. However, the specific mechanisms and molecular interactions by which these different functions of AIFM1 are connected in health and disease are not well resolved. For example, AIFM1 deficiency affects OXPHOS predominantly by lowering the amount of respiratory chain complex I \cite{RN4}. Other components were found to be affected in a tissue specific manner. In AIFM1 deficient patients \cite{RN5}, ablation of AIFM1 in skeletal and heart muscle affected cytochrome \emph{c} oxidase (COX) in addition to complex I, whereas in liver, deficiency of complexes I and V was observed \cite{RN10, RN9}.

In the present study, we explored the molecular interactions of AIFM1 with the multiprotein complexes of the OXPHOS system in heart mitochondria using our recently established complementary experimental approach (\textbf{\autoref{fig:ch3_fig1}}) \cite{RN11} that combines cross-linking mass-spectrometry (XL-MS) and complexome profiling. To increase the depth and confidence of the study, bovine heart mitochondrial membranes (BHM) were treated with three different chemical cross-linkers: DSSO \cite{RN12}, PhoX \cite{RN13} and DMTMM \cite{RN14}. While DSSO and PhoX predominantly generate lysine-lysine residue cross-links, DMTMM acts as a condensation reagent of acidic side chains of aspartic or glutamic acids with lysine side chains, resulting in the formation of a stable bond between those residues. We found that a significant fraction of AIFM1 in its dimeric form is specifically bound to monomeric cytochrome \emph{c} oxidase, an interaction that has been overlooked so far. By using the identified cross-links as structural restraints, we generated a structural model of dimeric AIFM1 docked to monomeric COX.

\begin{figure*}[htb]
    \center
    \includegraphics[]{Chapter.3/Figures/Figure1.png}
    \caption{\textbf{Two-tier experimental strategy for the analysis of proteome-wide protein-protein interactions in bovine heart mitochondria.} Mitochondria membranes were cross-linked with either of the three cross-linking reagents DSSO, PhoX or DMTMM. Subsequently, samples were analyzed by cross-linking mass spectrometry (XL-MS) and complexome profiling. Identified cross-linked peptides were used to generate protein-protein interaction networks. Protein interactions and structural models of AIFM1 with COX were then computationally modelled using the distance restraints from XL-MS data together with the assembly state and stoichiometry information obtained by complexome profiling.}
    \label{fig:ch3_fig1}
\end{figure*} \clearpage
%
\section{Results and Discussion}
We analyzed the organization and interaction landscape of protein complexes in bovine heart mitochondria by combining cross-linking mass spectrometry XL-MS and complexome profiling \cite{RN15, RN11} thereby adding new information on native state multiprotein complexes of interest and expanding previous work that explored the interactome of mitochondria from different organisms, tissues and cells by XL-MS \cite{RN20, RN18, RN16, RN19, RN17}.
%
\subsection*{Exploring mitochondrial complexes by combined cross-linking and complexome profiling}
To increase the depth of the protein-interaction map of BHM we applied multiple cross-linkers (DSSO, PhoX and DMTMM). Throughout the manuscript, the term cross-link is used to describe a link between two residues coming from two unique peptides, with an intra cross-link describing a linked residue pair within a protein and an inter cross-link describing a linked residue pair between two different proteins. Additionally, identified cross-links were filtered, so that only cross-links corresponding to protein-protein interactions that were reported for at least two cross-linkers and with at least two CSMs were kept. Covering 215 proteins listed in MitoCarta 3.0 \cite{RN21}, we obtained a total of 4413 unique cross-links (3261 intra- and 1152 inter-protein cross-links; \textbf{\autoref{fig:ch3_app_fig1}A}, \textbf{Dataset S1}). In accordance with previously published studies \cite{RN23, RN22}, the abundance of detected cross-linked proteins was higher than the median of all identified proteins in the BHM sample (8.8 vs. 6.9 log\textsubscript{10} iBAQ; \textbf{\autoref{fig:ch3_app_fig1}B}). Reflecting the large number and high abundance of membrane integral multiprotein complexes and the very high protein density, especially within the inner mitochondrial membrane, $\sim$75\% of the cross-links identified involved membrane proteins (\textbf{\autoref{fig:ch3_app_fig1}C}). For the same reasons and corroborating previous studies using mouse and human mitochondria \cite{RN16, RN21, RN19, RN17} the largest number of cross-links reflected interactions between the many subunits of OXPHOS complexes and their association to supercomplexes of respiratory chain complexes I, III and IV (1431 out of 4131 cross-links; \textbf{\autoref{fig:ch3_app_fig1}D}), also called respirasomes \cite{RN24}. Further, inter domain cross-links for complex I-III and COX where in very good agreement with previously published structural models, but providing no indications for homo-dimerization (complex I, II and COX), or multimerization (complex III; \textbf{Dataset S1}). In contrast, a significant portion of inter domain cross-links for complex V showed substantially more apparent restrain violations (\textbf{Dataset S1}). Most likely, this reflected the formation of higher order ATPase assemblies involved in shaping tightly curved cristae ridges \cite{RN25, RN26, RN27}, which can also be observed in the complexome profiling data (\textbf{\autoref{fig:ch3_app_fig1}E}).

Complexome profiling analysis of untreated (i.e. non-cross-linked) BHM yielded very similar results as those obtained previously with rat heart mitochondria using the same approach \cite{RN15} showing very similar migration pattern of the OXPHOS complexes and respirasomes (\textbf{\autoref{fig:ch3_app_fig1}E}; \textbf{Dataset S2}). When the samples were cross-linked with PhoX and DMTMM before subjecting them to complexome profiling, the overall abundance of detected proteins was not affected substantially. However, it was evident from the migration profiles of OXPHOS complexes that cross-linking to some extent prevented dissociation of complex V (CV) dimers and other fragile higher order respiratory supercomplexes during native electrophoresis (\textbf{\autoref{fig:ch3_app_fig1}E}). Importantly, in most cases the apparent molecular masses of the bulk of the OXPHOS complexes were not markedly affected by cross-linking. A notable exception was complex III (CIII) in the DMTMM treated sample, where the obligatory dimer did not migrate predominantly at the predicted apparent mass of $\sim$500 kDa as in all other conditions, but appeared at $\sim$650 kDa and showed multiple peaks at higher masses. The shift of the CIII dimer to higher masses suggested that, possibly through the large hydrophilic domains of its two core subunits, this OXPHOS complex cross-linked to a much larger extent to other mitochondrial proteins than the others. The $\sim$800 kDa peak corresponds to a previously described supercomplex between one complex III dimer and one complex IV (COX) monomer \cite{RN28}. The latter was also found in untreated and PhoX cross-linked samples, but was much more pronounced after cross-linking with DMTMM. The peaks at $\sim$1100 kDa and $\sim$1300 kDa can be interpreted as dimers of complex III dimers without and with one monomer of COX, respectively.

Taken together, these results establish that classical XL-MS analysis alone and in combination with complexome profiling delivered consistent results. Separating native complexes prior to mass spectrometric analysis provided additional key information on their apparent molecular masses and multimeric state. Cross-linking them beforehand allowed for more reliable detection of more fragile assemblies that otherwise partially or completely dissociate during solubilization and/or electrophoresis.

\subsection*{A specific complex between AIFM1 dimers and COX revealed by XL-MS}
When we performed an in-depth analysis of all detected cross-links involving OXPHOS complexes in addition to engaging their canonical components themselves, one specific protein stood out: In all our XL-MS datasets combined, AIFM1 had inter cross-links with no less than six subunits of COX with 82\% of them involving COX6B1 and COX6C (\textbf{\autoref{fig:ch3_fig2}A}; \textbf{Dataset S3}). Cross-links with COX subunits accounted for 86\% of the total inter cross-links with AIFM1. Adenylate kinase 2 (AK2) and adenine nucleotide carrier isoform 1 (SLC25A4) were the only other two proteins featuring multiple inter-protein cross-links with AIFM1.

\begin{figure*}[htb!]
    \center
    \includegraphics[]{Chapter.3/Figures/Figure2.png}
    \caption{\textbf{Dimeric AIFM1 forms a defined complex with monomeric COX.} \textbf{A.} Interaction network of AIFM1 in cross-linked BHM. Bold numbers indicate the observed cross-links for each interaction and thickness of lines indicate the cumulative evidence (CSMs) for each interaction (number in parentheses). Orange lines indicate cross-links involving AIFM1, while cross-links between AIFM1 interactors are presented as gray lines. \textbf{B.} Xi-net plot of the COX-AIFM1 interaction. Purple colored links indicate intra cross-links. Green colored links indicate inter cross-links. Respective sequence and cross-link features are indicated accordingly. \textbf{C.} Migration profiles of AIFM1 (orange) and averaged COX (green) from non-cross-linked (untreated) and cross-linked (PhoX or DMTMM) mitochondria separated by BN-PAGE (4-16\%). Peaks are annotated based on the apparent molecular mass of AIFM1 ($\sim$62 kDa) and monomeric COX ($\sim$220 kDa). In all samples, peaks corresponding to monomeric AIFM1 and COX as well as a peak corresponding to a COX-AIFM1\textsubscript{2} complex are observed. Although already present in the non-cross-linked sample, treatment with DMTMM seems to somewhat stabilize the COX-AIFM1\textsubscript{2} complex.}
    \label{fig:ch3_fig2}
\end{figure*}
The association of AIFM1 with this OXPHOS complex is remarkable in particular, since COX from bovine heart is undoubtedly the longest and best studied version of cytochrome \emph{c} oxidase \cite{RN29}. Therefore, we interrogated an earlier cross-linking dataset of mouse heart mitochondria for this interaction \cite{RN16}. Corroborating our findings, the majority of AIFM1 cross-links identified in this study engaged three different COX subunits, with COX6C being the most prominent by far (\textbf{\autoref{fig:ch3_app_fig2}A}). Of note, Liu and coworkers detected multiple cross-links between AIFM1 and AK2 as well in mouse heart mitochondria. Moreover, charting large affinity purifications-mass spectrometry (AP-MS) depositories, we found that they contained multiple instances of COX subunits interacting with AIFM1 \cite{RN30, RN32, RN31} (\textbf{Dataset S2}). Yet, buried in datasets generated by large-scale analyses of the mitochondrial interactome, these indications for AIFM1 binding to COX seem to have gone unnoticed so far.

Detailed evaluation of the observed cross-links between AIFM1 and COX \linebreak (\textbf{\autoref{fig:ch3_fig2}B}; \textbf{\autoref{fig:ch3_app_fig2}B}) revealed that they were predominantly within the pyridine nucleotide-disulfide oxidoreductase domain (Pfam: PF07992; res 136-460) of AIFM1 comprising one FAD- and one NADH-binding domain. Suggesting that AIFM1 had not been cleaved to its truncated pro-apoptogenic form \cite{RN33}, additional intra- and inter-protein cross-links were observed at the N-terminal end of the pro-peptide (res 55-101) of AIFM1 that is predicted to cross the inner mitochondrial membrane (IMM) reaching to the matrix side. Notably, these cross-links were the only ones to the matrix facing subunit COX5A, while all other cross-links engaged domains of COX subunits facing the intermembrane space (IMS).

Our three independent cross-linking analyses strongly suggested that AIFM1 and COX formed a specific complex, but provided no information on the multimeric state of the interaction partners and how much of this unexpected complex was present in bovine heart mitochondria. Therefore, we applied complexome profiling to analyze complexes containing AIFM1 and COX using, the same samples as in the XL-MS analysis (\textbf{\autoref{fig:ch3_fig2}C}; \textbf{Dataset S2}). In all cases, COX was predominantly present as a monomer ($\sim$220 kDa) and a prominent fraction of AIFM1 was found to migrate at an apparent mass consistent with its monomeric state (62 kDa). Substantial amounts of AIFM1 dimers were only observed in untreated BHM indicating that they may be destabilized by the cross-linking protocol. This was possibly due to partial oxidation of NADH known to be required for AIFM1 dimerization \cite{RN8}. Importantly however, a significant amount of AIFM1 consistently showed up as a peak at an apparent mass of $\sim$350 kDa in untreated mitochondria as well as after cross-linking with PhoX or DMTMM. This peak coincided with a shoulder next to the prominent peak at $\sim$220 kDa of monomeric COX in all samples analyzed, thus suggesting the presence of a $\sim$350 kDa complex containing a dimer of AIFM1 ($\sim$124 kDa) bound to monomeric COX ($\sim$220 kDa). Notably, a shoulder on the higher mass side of the COX monomer can also be observed in complexome profiling data of human cells published earlier, but its significance was not evident at the time \cite{RN34}. Label free quantification revealed that hardly any of the other respiratory chain complexes were present in this segment of the migration profiles. In contrast, the amounts of the COX monomer and AIFM1 dimer were comparable at $\sim$350 kDa suggesting a stoichiometric association and reflecting the observations for gels with increased resolution (high range BN-gel (3-10\%); \textbf{\autoref{fig:ch3_app_fig2}C}). At the same time, no AIFM1 co-migrated with the bulk of monomeric COX at $\sim$220 kDa (\textbf{\autoref{fig:ch3_app_fig2}D}). Notably, only small amounts of AIFM1 were detected at $\sim$1,850 kDa, the predicted mass of supercomplex S1 (I\textsubscript{1}III\textsubscript{2}IV\textsubscript{1}). This was mostly observed in the DMTMM treated sample that exerted many more inter-protein cross-links in the high mass range overall (\textbf{\autoref{fig:ch3_app_fig1}A,D}). It can be concluded that in our samples, AIFM1\textsubscript{2} was bound almost exclusively to monomeric COX and, if any, very little could be found associated with supercomplexes. Consistent with its higher cross-linking efficiency, the fraction of COX engaged in the complex with AIFM1\textsubscript{2} was somewhat higher with DMTMM than in the untreated and PhoX cross-linked samples. In fact, in untreated samples the amount of COX-AIFM1\textsubscript{2} complex was variable to some extent. This suggested that it tended to dissociate during solubilization and native electrophoresis. Such behavior has been observed previously for several less tightly associated subunits of OXPHOS complexes \cite{RN36, RN37, RN35}. Overall, we could estimate that about 10\% of monomeric COX was engaged in a fairly stable stoichiometric complex with AIFM1 dimers (\textbf{\autoref{fig:ch3_app_fig2}D}).

In summary, combination of cross-linking and complexome profiling data provided compelling evidence for the presence of a defined COX-AIFM1\textsubscript{2} complex in bovine heart mitochondria. The interaction interface was defined as involving residues of the neighboring COX6C, COX6B1, NDUFA4, COX6A2 and MT-CO2 contacting the pyridine nucleotide-disulfide oxidoreductase domain of AIFM1, and residues of COX5A interacting with the matrix facing N-terminal region of its pro-peptide.

It should be noted that cross-links between AIFM1 and the complex I subunit NDUFA8 were previously reported in a study by Liu \emph{et al.} \cite{RN16}, which may relate to the small amounts of AIFM1 found in the supercomplex range. However, inspection of the high-resolution supercomplex structure (PDB: 5XTH, \cite{RN38}) revealed that the COX subunits cross-linking to AIFM1 (COX6B1, Mt-CO2, NDUFA4, COX6C1, COX5A, COX6A2) and complex I subunit NDUFA8 are too far apart from each other to be consistent with simultaneous binding of the two OXPHOS complexes to AIFM1 within the respirasome. This is fully in line with our cross-link data and proposed model of the complex.
%
\subsection*{Creation of a cross-link guided structural model of the COX-AIFM1\textsubscript{2} complex}
Next, we aimed at building a structural model for the COX-AIFM1\textsubscript{2} complex guided by the distance restraints obtained by cross-linking, also including those involving the N-terminal sequence of AIFM1 comprising its pro-peptide sequence (res 55-101). We first derived a de novo model of this so far structurally unresolved region using trRosetta \cite{RN39} to complement a homology model of the bovine AIFM1 dimer that we derived from the human structure (PDB: 4BUR; \cite{RN40}) using Robetta \cite{RN41}. The structural model obtained for the N-terminal domain of AIFM1 agreed well with secondary structure predictions and featured three alpha-helices (res 67-88; 94-98; 105-112) of which the first is predicted to be a transmembrane segment (\textbf{\autoref{fig:ch3_app_fig3}A}). We then used restraints derived from our cross-linking data to dock this model of the bovine AIFM1 dimer to the 1.8 Å structure of COX (PDB: 1V54) \cite{RN42}. Unfortunately, this COX structure does not contain the more loosely attached NDUFA4 subunit. Therefore, we used Robetta \cite{RN41} to complement it with a homology model derived from the human NDUFA4 structure (PDB: 5Z62 chain N)  \cite{RN43}.

Mapping the cross-links onto the structural models of AIFM1\textsubscript{2} and COX respectively, revealed that the majority of cross-links were below 30 Å, with a combined mean distance of 19.1 Å for DSSO/PhoX and 20.8 Å for DMTMM cross-links (\textbf{\autoref{fig:ch3_app_fig3}B-C}; \textbf{Dataset S3}). The mean distance for DSSO and PhoX cross-links was well within the theoretical maximal range of $\sim$30 Å and $\sim$25 Å, respectively. DMTMM cross-links averaged somewhat above the theoretical maximum of $\sim$15 Å, in line with previous observations \cite{RN14}. Note, that eight cross-links for AIFM1 and ten cross-links for COX were not included in these calculations, because they involved intra cross-links from AIFM1 (res 128-613) to its de novo modelled N-terminus (res 55-124) or regions not resolved in the structural models (AIFM1 res 517-550; COX4l1 res 23-25; \textbf{\autoref{fig:ch3_app_fig3}C}).

Based on solvent accessibility and distance restraints obtained from both structures, accessible interaction interfaces between COX and the AIFM1 dimer as well as COX and the N-terminal region of AIFM1 were calculated using DisVis \cite{RN44}. While this analysis suggested that the AIFM1 dimer attaches to the intermembrane space side of COX, the predicted interaction space for the N-terminal region of one AIFM1 protomer covers the transmembrane domain at the matrix side of COX making contacts to subunits COX6B1, COX6C, MT-CO2, NDUFA4 (\textbf{\autoref{fig:ch3_fig3}A}). Scoring the interface models using the restraints imposed by the cross-linking data suggests that the COX-AIFM1\textsubscript{2} interface is mostly occupied by just one AIFM1 protomer. In agreement with this notion, cross-links suggested that only one N-terminal region, not both of the AIFM1 dimer interacted directly with COX. Therefore, the final modelling of the COX-AIFM1\textsubscript{2} complex was performed by docking just one AIFM1 protomer and one N-terminal region of AIFM1 to the COX monomer. Docking was restraint by 46 of 59 unique cross-links detected for the COX-AIFM1 interaction. Haddock \cite{RN45} generated six acceptable clusters (with negative Haddock score). One cluster (cluster 7) showed better scores (overall lowest Haddock score and i-RMSD) compared to the other clusters (\textbf{\autoref{fig:ch3_app_fig4}A}). Investigation of the individual structures of produced clusters revealed that only for structures of cluster 7 the N-terminal domain of AIFM1 was positioned in accordance with its transmembrane domain. Furthermore, structural alignment of models from cluster 7 showed high cluster precision and only differed marginally from each other (RMSD = 0.942 Å; \textbf{\autoref{fig:ch3_app_fig4}B}). Lastly, clusters were validated by mapping the used cross-link restraints on the highest scoring model of each cluster, with the structure of cluster 7 satisfying them best (\textbf{Dataset S4}).
\begin{figure*}[p]
    \center
    \includegraphics[]{Chapter.3/Figures/Figure3.png}
    \caption{\textbf{Cross-link derived structural model of the COX-AIFM1\textsubscript{2} complex.} \textbf{A.} Visualization of the cross-link driven accessible interaction space models for a COX-AIFM1\textsubscript{2} complex. COX is represented in green, while the bright orange volume represents the center-of-mass position of the AIFM1 dimer, and the dark orange volume represents the center-of-mass position of the model of the AIFM1-N-terminus (res 55-124). The cross-linking data are consistent with the interaction space available for docking dimeric AIFM1 and the N-terminal region of one AIFM1 protomer to monomeric COX. \textbf{B.} Cross-link derived structural model of the COX-AIFM1\textsubscript{2} complex. COX is represented in green, and AIFM1 protomers (res 128-516; 551-613 with and without N-terminal region (res 55-127) are represented in orange and yellow respectively. The transmembrane residues (67-85) of the N-terminus of the interacting AIFM1 moiety is highlighted in red. Membrane boundaries of the IMM are sketched as gray spheres. The final complex consists of monomeric COX, dimeric AIFM1 (res 128-516, 551-613) and the N-terminal region of one AIFM1 protomer (res 55-127).}
    \label{fig:ch3_fig3}
\end{figure*}
Based on overall best cluster scoring and high cluster precision, the highest scoring structure of cluster 7 was chosen as representative model for the COX-AIFM1\textsubscript{2} complex. In the final structural model of the COX-AIFM1\textsubscript{2} complex, the AIFM1 dimer “sits” on COX facing the intermembrane space side and makes contact through one AIFM1 protomer covering parts of COX6B1, COX6C1, MT-CO2 and NDUFA4 (\textbf{\autoref{fig:ch3_fig3}B}). The second AIFM1 protomer points away from COX making just one very limited contact to COX through its C-terminal loop. At the opposite side of COX, the N-terminal region of the interacting AIFM1 protomer makes contact with COX6B1 and transmembrane helices of MT-CO2 and NDUFA4 (\textbf{\autoref{fig:ch3_fig3}B}). It should be noted that after docking with Haddock, the de novo structural model covers the N-terminal region of AIFM1 only up to residue 124 creating a structurally undefined stretch of three amino acids up to residue 128, the first amino acid contained in the homology model for the main part of AIFM1. Therefore, residues 121 to 131 were re-modelled using the “Model Loops” interface in Chimera \cite{RN47, RN46}, thereby connecting the N-terminal domain to the main part of AIFM1. Cross-links (46) used for structural modelling as well as all observed cross-links (59) for COX-AIFM1\textsubscript{2} were in good agreement with the final structural model of COX-AIFM1\textsubscript{2} (\textbf{\autoref{fig:ch3_app_fig4}C-D}). The combined mean distance for DSSO/PhoX cross-links used for docking was slightly lower (26.5 Å, 20 cross-links) than for DMTMM (28.2 Å, 26 cross-links) (\textbf{Dataset S4}). Most of the obtained over length cross-links (> 33 Å, 15 cross-links) involve the N-terminal domain of AIFM1 (12 out of 15 cross-links). Interactions to the N-terminal domain are predominately mapped with DMTMM (20 out of 24 detected cross-links), explaining the slightly higher mean distance. The over length cross-links involving the AIFM1 N-terminus were obtained for cross-links to a flexible segment of COX6C (6 cross-links) and a defined residue stretch (221-244) of AIFM1 (6 cross-links) located closely to the AIFM1 N-terminus. These over length cross-links predominantly involving specific domains could indicate that cross-links are derived for several assemblies rather than one assembly, which is a challenge of in-solution XL-MS described previously \cite{RN11}. In our case, cross-links observed between AIFM1 and the AIFM1 N-terminus might well result also from monomeric or dimeric AIFM1 assemblies (\textbf{\autoref{fig:ch3_fig2}C}) potentially featuring different domain orientations as compared to AIFM1 in complex with COX. Such averaging likely explains the observed over-length cross-links. Notwithstanding these considerations, the COX-AIFM1 model is in good agreement with the cross-linking data.

Next, we performed an interaction interface analysis of the docking model that predicted three distinct interfaces between COX and AIFM1 (\textbf{\autoref{fig:ch3_fig4}A}; \textbf{Dataset S5}). The first, extensive interface is defined by the N-terminal residues of AIFM1, which interacts with neighboring residues of the COX subunits MT-CO2, NDUFA4 and COX6B1. Secondly, residues of the pyridine nucleotide-disulfide oxidoreductase domain of AIF\-M1 comprising the NADH- and FAD-binding domains intimately interact with MT-CO2, COX6B1 and COX6C. The third, rather small interaction interface is defined by residues of the C-terminal region of the second AIFM1 protomer and residues of MT-CO2, COX4l1 and COX7B (\textbf{\autoref{fig:ch3_fig4}B}). The interface between the hydrophilic parts of the N-terminal region of AIFM1 facing the intermembrane space is mostly driven by contacts to COX6B1 and NDUFA4, whereas its transmembrane domain predominantly interacts with one of the transmembrane segments of MT-CO2. Notably, the very N-terminal residues of AIFM1 facing the matrix side reside within a $\sim$25 Å distance from residues 44-54 of COX5A, consistent with the observed cross-links to this COX subunit (\textbf{\autoref{fig:ch3_fig2}B}).
\clearpage
%
\subsection*{Potential functional implications of a COX-AIFM1\textsubscript{2} complex}
While the predominant consequence of AIFM1 deficiency is impaired complex I assembly \cite{RN4}, additional COX deficiency has been reported in skeletal muscle and heart \cite{RN10, RN9}, as well as in \emph{Drosophila melanogaster} \cite{RN48} and \emph{Caenorhabditis elegans} \cite{RN49}. Conversely, AIFM1 expression was found to be significantly increased along with several COX assembly factors in human COX-negative muscle fibers \cite{RN50}. However, it seems unlikely that the COX-AIFM1\textsubscript{2} complex described here contributes to assembly or stabilization of COX, because it accounted only for 10\% or less of the total amount of this OXPHOS complex. Moreover, no apparent co-migration between AIFM1 and any of the individual COX subunits or sub-assemblies at apparent masses lower than $\sim$350 kDa was observed, suggesting that association of AIFM1 occurred only with fully assembled COX. Conditional involvement of AIFM1 in the maturation of COX assembly factors that are substrates of the disulfide relay of the intermembrane space \cite{RN8, RN7, RN6} appears as a more likely explanation for the link between AIFM1 and COX deficiency in some tissues.

It has been reported that AIFM1 is a member of the NDH-2 family of proteins \cite{RN51} and thus exhibits NADH:ubiquinone oxidoreductase activity \cite{RN52}. Nevertheless, we examined the possibility of direct electron transfer between the FAD and CuA of COX within the COX-AIFM1\textsubscript{2} complex. The minimal distance between the isoalloxazine moieties of the FADs and the CuA center was $\sim$50 Å and $\sim$55 Å (\textbf{\autoref{fig:ch3_app_fig5}}), which is more than three-times larger than the 14 Å considered as the maximum distance for efficient electron tunneling in a protein matrix \cite{RN53}. It would be conceivable that this distance is bridged by cytochrome \emph{c} (CytC) serving as an electron shuttle between AIFM1 and COX. Therefore, we explored whether cytochrome \emph{c} could still bind to its substrate binding site in the COX-AIFM1\textsubscript{2} complex. Merging our structural model with a previously obtained model of cytochrome \emph{c} bound to COX from bovine heart \cite{RN54} suggested that the AIFM1 dimer does not hamper cytochrome \emph{c} from binding to COX (\textbf{\autoref{fig:ch3_fig4}C}). However, distances of $\sim$45 Å and $\sim$47 Å between the heme moiety of cytochrome \emph{c} and the isoalloxazine rings of FAD in both AIFM1 protomers excluded direct electron transfer also in the presence of the additional heme. Yet, a substrate channeling mechanism could still be imaginable implying movement of cytochrome \emph{c} forth and back between AIFM1 and COX without leaving the complex. A crevice in the second AIFM1 protomer facing COX could potentially reduce the distance between the redox centers indeed to about 14 Å. However, this would require cytochrome \emph{c} to turn within the pocket formed by AIFM1\textsubscript{2} and COX in order to bring its heme as close as possible to the isoalloxazine ring. Thus, while such a substrate channeling mechanism cannot be excluded, it does not seem very likely. Moreover, oxidizing NADH would transiently destabilize dimerization of AIFM1 \cite{RN8} and thus the entire complex, arguing further against any oxidoreductase activity of the COX-AIFM1\textsubscript{2} complex.

Since our structural model excludes electron transfer from AIFM1 to COX, it seems unlikely that the complex between them serves to drain electrons from the disulfide relay of the intermembrane space by regenerating CHCHD4/MIA40 \cite{RN6}. Therefore, it remains to be established, whether there is any functional link between the COX-AIFM1\textsubscript{2} complex described here and the import machinery for proteins of the mitochondrial intermembrane space containing disulfide bonds.

\begin{figure*}[htb!]
    \center
    \includegraphics[]{Chapter.3/Figures/Figure4.png}
    \caption{\textbf{Deciphering interaction interfaces in the COX-AIFM1\textsubscript{2} structural model.} \textbf{A.} Three distinct interfaces between COX subunits and respective AIFM1 protomers were found. Subunits (COX) and protein domains (AIFM1) with residues in respective interfaces are colored in gray. Active COX residues are shown as green colored sticks and active AIFM1 residues as orange colored sticks. \textbf{B.} Analysis of the number of residue contacts between respective COX subunits and AIFM1 domains. Colored circles indicate residue contacts between single subunits (COX) and domains (AIFM1) with the size of each circle corresponding to the number of residue-residue interactions. \textbf{C.} COX-AIFM1\textsubscript{2} complex with cytochrome \emph{c} (purple) bound to its COX binding site. The structural model presented here was merged with a previously published model of cytochrome \emph{c} docked to COX from bovine heart \cite{RN54}. COX subunits are colored green, while AIFM1 protomers are colored orange and yellow. The transmembrane (TM) domain of the N-terminal domain of AIFM1 is highlighted in red. Boundaries of the IMM are indicated as gray spheres.}
    \label{fig:ch3_fig4}
\end{figure*}
If COX-AIFM1\textsubscript{2} is not a catalytic complex, it is still tempting to speculate that a ternary interaction of COX, AIFM1\textsubscript{2} and cytochrome \emph{c} could play a role in mitochondrial pro-apoptotic mechanisms. Apart from directly promoting programmed cell death \cite{RN55, RN56, RN5}, AIFM1 could play an indirect role in apoptosis by modulating release of cytochrome \emph{c} \cite{RN1} through its binding to the COX-AIFM1\textsubscript{2} complex. For this, it is important to note that cytochrome \emph{c} makes direct contact to the first AIFM1 protomer in the ternary complex (\textbf{\autoref{fig:ch3_fig4}C}). It is important to note that a structure of bovine COX obtained by X-ray crystallography with its substrate bound \cite{RN57} showed CytC in a position that is different from the one suggested by the model of Sato et al. \cite{RN54} and that would not allow simultaneous binding of AIFM1. However, this apparent discrepancy is not unexpected, since different binding modes of CytC to COX were reported a long time ago based on chemical modification and kinetic studies \cite{RN58} and were confirmed recently by cryoEM analysis \cite{RN59}. Thus, different enzyme-substrate binding modes of CytC including the ones suggested by the docking model and the crystal structure may be physiologically relevant. In any case, the model by Sato et al. \cite{RN54} indicates that it is possible for CytC to bind in a position to COX that allows the formation of a ternary complex with AIFM1. Providing further support for this hypothetical ternary complex, cross-links between AIFM1 and cytochrome \emph{c} were previously reported in intact mouse heart mitochondria \cite{RN16}. However, it is also known that binding of cytochrome \emph{c} to COX is strongly reduced at higher ionic strength \cite{RN60}, which could explain, why we did not observe cross-links between cytochrome \emph{c} and the COX-AIFM1\textsubscript{2} complex. Since the N-terminal pro-peptide with its transmembrane helix provides a significant portion of the AIFM1/COX interface, the complex is expected to destabilize upon cleavage of AIFM1 thereby activating its pro-apoptotic function. In addition to cleaved AIFM1, any previously bound cytochrome \emph{c} would be released concomitantly further promoting apoptosis, potentially providing a synergistic boost to the cell death program already underway.
%
\section{Conclusion}
We show that $\sim$10\% of monomeric COX in bovine heart mitochondria are engaged in a defined complex with dimeric AIFM1. Using structural restraints provided by cross-linking data, available high-resolution structures and structural modeling, we could derive a model of the COX-AIFM1\textsubscript{2}  complex with and without bound cytochrome \emph{c}. Combining chemical cross-linking and complexome profiling provided useful complementary information and represents proof-of-concept for our experimental approach demonstrating that it can be used to define and characterize multiprotein assemblies in detail that may have been overlooked by other means.

While our structural model excludes direct electron transfer between AIFM1 and COX, it provides clues on potential functional implications of the formation of the COX-\textsubscript{2} complex including a possible involvement in promoting apoptosis. The structural insights into this unexpected mitochondrial complex will stimulate and guide further studies on the role of AIFM1 in OXPHOS biogenesis and apoptosis.
%
\section{Materials and Methods}
%
\subsection*{Isolation and purification of bovine heart mitochondria (BHM)}
Mitochondrial membranes from bovine heart were isolated and preserved as described in \cite{RN11}. In order to increase the purity of the preparation and for Tris-buffer removal, frozen crude mitochondria (4 x 15 ml aliquots; 60 mg protein/ml) were thawed on ice, diluted (1:4) with ice-cold SEH buffer (250 mM sucrose, 1 mM EDTA, 20 mM HEPES, pH 7.4 adjusted with NaOH) and centrifuged at 1,000 x g (10 min; 4°C). The supernatants were recovered and centrifuged at 40,000 x g (20 min; 4°C) and each resulting pellet was suspended in 2 ml SEH buffer. Afterwards, mitochondria were loaded onto a two-layer sucrose gradient (1 M sucrose, 20 mM HEPES, pH 7.4 /1.5 M sucrose, 20 mM HEPES, pH 7.4) and centrifuged at 60,000 x g (20 min; 4°C). The pure mitochondrial fractions accumulated at the interphase were carefully recovered and pooled into one tube. After resuspension in 20 ml ice-cold SEH buffer, pure mitochondria were centrifuged at 10,000 x g (20 min; 4°C) and finally suspended in 5 ml ice-cold SEH buffer supplemented with protease inhibitor cocktail (SIGMAFAST™). Protein concentration was determined by the DC protein assay (Bio-Rad) and aliquots of pure mitochondria were shock-frozen in liquid nitrogen and stored at -80°C until use.
%
\subsection*{Cross-linking of BHM sample with DSSO, PhoX and DMTMM}
Purified bovine heart mitochondrial membranes were buffer exchanged into cross-linking buffer (10 mM HEPES pH 7.8, 1 mM EDTA, 1 mM EGTA, 10 mM NaCl, 150 mM KCl, protease inhibitor). After optimization of the cross-link reaction, $\sim$2 mg of BHM were either incubated with DSSO (0.5 mM freshly re-suspended in anhydrous DMSO; Thermo Fisher Scientific), PhoX (1 mM freshly re-suspended in anhydrous DMSO; made in-house) or DMTMM (10 mM freshly re-suspended in cross-linking buffer; Sigma-Aldrich) in 2 ml of cross-linking buffer at room temperature (RT). The cross-link reaction was quenched after 30 min by the addition of 50 mM Tris (1 M Tris buffer, pH 8.5) for additional 30 min at RT.
%
\subsection*{Sample preparation for XL-MS analysis of cross-linked BHM}
Cross-linked mitochondria were solubilized with Digitonin (9 g/g protein) for 30-60 min on ice. Proteins were denatured and purified as described previously \cite{RN61}. Briefly, denatured proteins were re-suspended and digested overnight (ON) at 37°C with Lys-C followed by Trypsin. The final peptide mixtures were desalted with solid-phase extraction C18 columns (Sep-Pak, Waters). Samples cross-linked with DSSO and DMTMM were fractionated with an Agilent 1200 HPLC pump system (Agilent) coupled to an strong cation exchange separation column (Luna SCX 5 $\mu$m - 100 Å particles, 50 $\times$ 2mm, Phenomenex), resulting in 24 fractions . For PhoX cross-linking we used a Fe3+-IMAC column (Propac IMAC-10 4 $\times$ 50 mm column, Thermo Fisher scientific) connected to an Agilent HPLC. Lyophilized peptides were dissolved in buffer A (30\% acetonitrile, 0.07\% trifluoroacetic acid) and the pH was adjusted to a value of 2. PhoX cross-linked peptides were subsequently eluted with a gradient of elution buffer B (0.3\% NH4OH) \cite{RN62}. The collected PhoX-enriched peptides were then dried down and further fractionated into 7 high-pH fractions as previously described \cite{RN63}.
%
\subsection*{XL-MS analysis and data analysis}
The 24 SCX fractions of DSSO were injected in an Agilent 1290 Infinity UHPLC system (Agilent) on a 50-cm analytical column packed with C18 beads (Dr Maisch Reprosil C18, 3 $\mu$m) coupled online to an Orbitrap Fusion Lumos (Thermo Fisher Scientific). We used the following LC-MS/MS parameters: after 5 minutes of loading with 100\% buffer A (water with 0.1\% formic acid), peptides were eluted at 300 nL/min with a 97 minutes gradient from 4\% to 39\% of buffer B (80\% Acetonitrile and 20\% water with 0.1\% formic acid). For MS acquisition we used a MS1 Orbitrap scan at 120,000 resolution from 310 to 1600, AGC target of 5e\textsuperscript{5} ions and maximum injection time of 50 ms. The ions with a charge from +3 to +8 were fragmented with CID (NCE of 30\%) and analyzed with MS2 Orbitrap at 30,000 resolution, AGC target of 5e\textsuperscript{4} ions and maximum injection time of 54 ms for detection of DSSO signature peaks (difference in mass of 37.972 Da). The four ions with this specific difference were analyzed with a MS3 Ion Trap scans (AGC target of 2e\textsuperscript{4} ions, maximum injection time of 150 ms) for sequencing the individual peptides. For the fractions of DMTMM and PhoX, we used an Ultimate3000 (Thermo Fisher Scientific) and 50-cm analytical column packed with C18 beads (Dr Maisch Reprosil C18, 3 $\mu$m) heated at 45°C, connected to Orbitrap Fusion Lumos. For both experiment we used a gradient from 9\% to 40\%, but in case of DMTMM it was 90 minutes long while for PhoX 30 minutes. For both experiments, we used a MS1 Orbitrap scan at 120,000 resolution from 350 to 1400, AGC target of 1e\textsuperscript{6} ions and maximum injection time of 50 ms. The most abundant ions with a charge between +3 and +8 were fragmented in HCD (stepped NCE of 30±3\%) and analyzed with MS2 Orbitrap scan at 30,000 resolution, AGC target of 1e\textsuperscript{5} ions, and maximum injection time of 120 ms. The DSSO fractions were analyzed with Proteome Discoverer software suite version 2.4.1.15 (Thermo Fisher Scientific) with the incorporated XlinkX node for analysis of cross-linked peptides as reported by Klykov et al. \cite{RN64}. Data were searched against a FASTA file containing the $\sim$4200 most abundant proteins, which were previously determined following a classical bottom-up workflow. Were applicable, mitochondrial target peptides were removed from respective protein sequences. For XlinkX search, we selected fully tryptic digestion with three maximum missed cleavages, 10 ppm error for MS1, 20 ppm for MS2 and 0.5 Da for MS3 in Ion Trap. For modifications, we used static Carbamidomethyl (C) and dynamic Oxidation (M). The cross-linked peptides were accepted with a minimum score of 40, minimum score difference of 4 and maximum FDR (controlled at PSM level for cross-linked spectrum matches) rate set to 5\%. Both non-cleavable cross-linkers were analyzed with pLink2 \cite{RN65} and the same FASTA used for DSSO. For PhoX, we manually added the cross-linker to the list (alpha/beta sites “[K”, linker composition C(8)H(3)O(5)P(1) mass of 209.971Da) and for both cross-linkers, the same parameter settings as described for XlinkX was used with following exceptions: no minimum score option and the FDR was calculated separately for intra and inter cross-links. Finally, cross-links were additionally filtered: only cross-links corresponding to protein-protein interactions that were reported for at least two cross-linkers and with at least two CSMs were kept for the final interaction analysis and structural modeling.
%
\subsection*{Complexome profiling analysis}
Aliquots of untreated and PhoX and DMTMM cross-linked mitochondrial membranes (see “Cross-linking of BHM sample with DSSO, PhoX and DMTMM” for details) were thawed on ice, solubilized with digitonin (9 g/g protein) in 50 mM NaCl, 50 mM imidazole-HCl, 2 mM 6-aminohexanoic acid, 1 mM EDTA, pH 7 and kept on ice for 20 min. Samples were further centrifuged at 22,000 $\times$ g (20 min; 4°C) and the supernatants were transferred into clean tubes and supplemented with Coomassie blue loading dye as described in I. Wittig, et al. \cite{RN66}. For blue-native (BN)-PAGE, 100 $\mu$g protein of each sample were loaded onto 4-16\% or 3-10\% polyacrylamide gradient gels and separated as described previously \cite{RN66}. After the electrophoretic run, the gel was fixed ON in 50\% methanol, 10\% acetic acid, 100 mM ammonium acetate followed by staining with 0.025\% Coomassie blue G-250 (Serva G) in 10\% acetic acid for 30 min, de-stained twice in 10\% acetic acid (1 h each) and kept in deionized water ON. The next day, the gel was color-scanned using a flatbed Image Scanner III (GE, USA) to use it as a template for the cutting procedure. Proteins were identified by LC-MS/MS after in-gel tryptic digestion following the protocol described in H. Heide, et al. \cite{RN15} with some modifications. In short, each gel lane was cut into 60 even slices starting at the bottom of the gel. The slices were cubed and transferred into 96-well filter plates (Millipore\textsuperscript{®}, MABVN1250) adapted manually to 96-well plates (MaxiSorpTM Nunc) as waste collectors. Gel pieces were incubated with 50\% methanol, 50 mM ammonium hydrogen carbonate (AHC) under moderate shaking; the solution was refreshed until the blue dye was removed completely. Removal of excess solution was done by centrifugation (1,000 $\times$ g, 15 s). In the next step, gel pieces were reduced with 10 mM dithiothreitol in 50 mM AHC for 1 h. After removing excess solution, 30 mM chloroacetamide in 50 mM AHC was added to each well, incubated in the dark for 45 min and removed. A short incubation step with 50\% methanol, 50 mM AHC was performed for gel pieces dehydration ($\sim$15 min). The latter solution was removed and gel pieces were dried for $\sim$30 min at RT. Later, 20 $\mu$l of 5 ng $\mu$l-1 trypsin (sequencing grade, Promega\textsuperscript{®}) in 50 mM AHC plus 1 mM CaCl2 were added to each well and incubated for 20 min at 4°C. Gel pieces were covered by adding 50 $\mu$l of 50 mM AHC followed by an ON incubation at 37°C for protein digestion. The next day, the peptide-containing supernatants were collected by centri¬fugation (1,000 $\times$ g, 30 s) into clean 96-well PCR plates (Axygen\textsuperscript{®}). The gel pieces were finally incubated with 50 $\mu$l of 30\% acetonitrile (ACN), 3\% formic acid (FA) for $\sim$30 min prior elution of the remain¬ing peptides on the previous eluates by centrifugation. The peptides were dried in a SpeedVac Concentrator Plus (Eppendorf) for 2.5-3 hours, resuspended in 20 $\mu$l of 5\% ACN, 0.5\% FA and stored at -20 °C until MS analysis. After thawing the frozen resuspended peptides and a 30 min gentle shaking, individual samples were loaded and separated by reverse phase liquid chromatography and analyzed by tandem mass spectrometry in a Q-Exactive Orbitrap Mass Spectrometer equipped with a nano-flow ultra-HPLC system (Easy nLC1000, Thermo Fisher Scientific). In brief, peptides were separated using 100 $\mu$m ID $\times$ 15 cm length PicoTipTM EMITTER columns (New Objective) filled with ReproSil-Pur C18-AQ reverse-phase beads (3 $\mu$m, 120Å) (Dr. Maisch GmbH, Germany) using linear gradients of 5\%-35\% ACN, 0.1\% FA (30 min) at a flow rate of 300 nl min-1, followed by 35\%-80\% ACN, 0.1\% FA (5 min) at 600 nl min-1 and a final column wash with 80\% ACN (5 min) at 600 nl min-1. All settings for the mass spectrometer operation were the same as detailed in \cite{RN34}.
MS raw data files from all individual slices were analyzed using MaxQuant (v1.5.0.25) against the \emph{Bos taurus} proteome entries retrieved from Uniprot. The following settings were applied: Trypsin, as the protease, N-terminal acetylation and methionine oxidation as variable modifications; cysteine carbamidomethylation as fixed modification; two trypsin missed cleavages; matching between runs, 2 min matching time window; six residues as minimal peptide length; common contaminants included; I = L and the rest of parameters were kept as default. Individual protein abundances were determined by label-free quantification using the obtained intensity-based absolute quantification (iBAQ) values, which were corrected for protein loading and MS sensitivity variations using the sum of total iBAQ values from each sample. For each protein group entry, migration profiles were generated and normalized to the maximal abundance through all fractions. The migration patterns of the identified proteins were hierarchically clustered by an average linkage algorithm with centered Pearson correlation distance measures using Cluster 3.0 \cite{RN67}. The resulting complexome profiles consisting of a list of proteins arranged according to the similarity of their migration patterns in BN-PAGE were visualized as heatmaps representing the normalized abundance in each gel slice by a three-color gradient (black/yellow/red) and processed in Microsoft Excel for analysis. The mass calibration for the BN gel was performed using the apparent molecular masses of either membrane or soluble bovine heart mitochondrial proteins. For membrane proteins: VDAC1 (30 kDa), complex II (123 kDa), complex IV (215 kDa), complex III (dimer, 485 kDa), complex V (700 kDa), complex I (1000 kDa), respiratory supercomplexes, I-IV (1215 kDa), I-III2 (S0, 1485 kDa), I-III2-IV (S1, 1700 kDa), I-III2-IV2 (S2, 1915 kDa) and complex V tetramer (2400 kDa). For soluble proteins: ATP synthase subunit beta (51 kDa), citrate synthase (dimer, 98 kDa), ETFA/B (dimer, 122 kDa), enoyl-CoA hydratase (hexamer, 169 kDa), fumarase (tetramer, 200 kDa), Heat shock protein 60 (heptamer, 406 kDa), PCCA/B (hexamer 762 kDa) and oxoglutarate dehydrogenase complex ($\sim$2500 kDa).
%
\subsection*{Generation of structural models for COX and AIFM1}
Firstly, as no structure of bovine (dimeric) AIFM1 is currently available, a homology model was generated and structurally aligned based on the human dimeric AIFM1 structure (PDB: 4BUR) using Robetta. The final dimeric model of AIFM1 lacks the N-terminal region, containing residues 128-516 and 551-611 for both molecules. The N-terminal region of AIFM1 (res 55-124) was generated using trRosetta \cite{RN39}. Further, a monomeric COX structure was generated from the recently published bovine COX dimer (PDB: 1V54). The structure was modified by adding the missing NDUFA4 subunit, which was modelled and structurally aligned based on the human homolog (PDB: 5Z62 chain N) using Robetta. Likewise, missing residues (without transit peptides) for COX6B1 and COX5A subunits were modelled and added to the COX and added to the final COX structure used for docking.
%
\subsection*{Cross-linking driven docking and analysis of a COX-AIFM1\textsubscript{2} complex}
To generate a COX-AIFM1 structure, modified structures for COX, AIFM1 and the N-terminal domain of AIFM1 were used. Firstly, interaction interfaces and cross-links supporting a distinct complex formation were identified using DisVis \cite{RN44}. Active residues involved in an interface were computed additionally based on solvent accessible residue information. Solvent accessible residues (absolute and relative solvent accessibility $\geq$ 40\%) were identified using the standalone program Naccess (© S. Hubbard and J. Thornton 1992-6). Structural docking with respective structures was done in Haddock \cite{RN68, RN45} using predicted active residues and cross-links as additional restraints. Different distance allowances between C$\alpha$-C$\alpha$ atoms were used based on the observed cross-link: DSSO = 35 Å, PhoX = 30 Å and DMTMM = 25 Å. Docking of COX, the AIFM1 dimer and one N-terminal domain of AIFM1was performed separately, to ensure correct positioning of the N-terminal domain of AIFM1 with respect to COX and the respective AIFM1 protomer. The missing residues of the flexible linker between the AIFM1 protomer and the N-terminal domain were built afterwards to complete the structure and to further verify the positioning of the N-terminal domain. Missing residues for cross-linked subunits of COX involved in the COX-AIFM1 interaction were modelled before the docking procedure. The finally chosen model was the best scoring model within the best scoring cluster which also supported the cross-linking restraints best. Subsequently, the “Model Loops” of Chimera (Version 1.14rc) \cite{RN47, RN46} was applied to model missing residues (125-127) and structurally connecting the N-terminal domain of AIFM1 (res 55-124) and respective AIFM1 protomer (28-516; 551-613). Interface residues of the resulting COX-AIFM1\textsubscript{2}  complex were identified using the Prodigy web service \cite{RN69}. To determine whether cytochrome \emph{c} can still potentially bind to its COX binding site in the COX-AIFM1\textsubscript{2}  complex, this protein was structurally aligned based on a previously solved structure of cytochrome \emph{c} docked to COX from bovine heart \cite{RN54}. Presented membrane boundaries for all presented structures were added using either the OPM \cite{RN70} webserver or MemprotPD \cite{RN71}.
%
\subsection*{Data availability}
All cross linking mass spectrometry related data, the structural docking (Haddock results), and the presented COX-AIFM1/ COX-AIFM1-CytC structural model described in this work have been deposited to the ProteomeXchange partner (PRIDE) database and assigned the identifier PXD025102 \cite{RN72}. Complexome profiling datasets have been deposited in the CEDAR database and assigned the identifier CRX33 \cite{RN73}. The structural model of COX-AIFM1\textsubscript{2} complex is provided in the PDB-DEV repository (PDBDEV\_00000092). Supplementary Datasets are available online with the original manuscript (Supporting Information).
%
\subsection*{Acknowledgments}
All authors acknowledge support from the Netherlands Organization for Scientific Research (NWO) funding the Netherlands Proteomics Centre through the X-omics Road Map program (Project 184.034.019) and the TOP project 714.017.004, the Netherlands Organization for Health Research and Development (ZonMW TOP 91217009), the EU Horizon 2020 program Epic-XS (Project 823839), and the German Research Foundation (DFG) through the Collaborative Research Center 1218 (Project 2699\-25409).
%
\subsection*{Author contributions}
JFH, UB, SA and AJRH designed the study. JFH and RCZ performed XL-MS experiments. JFH performed XL-MS analysis and structural modeling. ACO performed the complexome profiling experiments. ACO, SA and UB analyzed the complexome profiling datasets. ACO and SA provided the bovine mitochondrial samples. JFH, SA and AJRH wrote the original draft, with all authors carefully revising and editing the manuscript before submission. UB, SA and AJRH acquired funding and resources. AJRH supervised the project.
%
\subsection*{Conflict of interest}
The authors do not declare any conflict of interest.
%
%%Appendix
\clearpage
\begin{subappendices}
    \counterwithin{figure}{section}
    \section{Supplementary Material}
    \begin{figure*}[hb!]
        \center
        \includegraphics[]{Chapter.3/Figures/SI_Figure1.png}
        \caption{Figure Legend on next page}
        \label{fig:ch3_app_fig1}
    \end{figure*}
    \addtocounter{figure}{-1}
    \begin{figure*}[ht!]
        \caption{\textbf{Overview of cross-linking and complexome profiling of bovine heart mitochondria (BHM).} BHM were cross-linked in parallel with three different cross-linkers (DSSO, PhoX and DMTMM) and subjected to XL-MS analysis or complexome profiling. \textbf{A.} Overview of the number of unique cross-links, identified for each of the cross-linkers used (DSSO, PhoX, DMTMM) and in combination (all). \textbf{B.} Observed cross-linked proteins are generally more abundant. Boxplot showing protein abundances for all identified proteins, all cross-linked proteins and proteins that have either only inter-, intra- or both (inter and intra) links. The number of proteins and the median iBAQ is indicated on top of each box. \textbf{C} Pie chart showing the sub-mitochondrial localization of the cross-linked proteins identified based on their MitoCarta 3.0 annotation. \textbf{D} Numbers of obtained cross-links for OXPHOS complexes I-V. Complexes are heavily cross-linked ($\sim$35\% of all detected cross-links). Cross-links are observed within subunits of the same complex (intra cross-links) but also between subunits of different OXPHOS complexes (inter cross-links). For structural representation, deposited structural models were chosen (PDB: 5LNK, 1ZOY, 1NTM, 1V54, 5ARA). \textbf{E} Averaged migration profiles of the OXPHOS complexes CI, CII, CIII and CV without cross-linker treatment and after treatment with either the PhoX or DMTMM cross-linker using a 4-16\% gradient BN gel. The profiles were obtained by plotting the relative abundance of the averaged subunits of each complex against the respective molecular mass. Peaks are annotated based on the molecular masses of CI, CII, CIII, CV and the supercomplex S1 (CI-CIII\textsubscript{2}-CIV). Upon addition of cross-linker, the OXPHOS complexes largely maintain their overall migration profile, and thus structural integrity.}
    \end{figure*}
    \begin{figure*}[htb]
        \center
        \includegraphics[]{Chapter.3/Figures/SI_Figure2.png}
        \caption{\textbf{COX-AIFM1 interactions.} \textbf{A.} The AIFM1 interactome as observed by XL-MS in intact mouse heart mitochondria adapted from \cite{RN16}. Bold numbers indicate the observed cross-links for each interaction and thickness of lines indicates the cumulative evidence (CSMs) for each interaction (number in brackets). CSMs were summed from triplicates. \textbf{B.} Visualization of cross-linked residues (red spheres) of COX (gray) and truncated AIFM1 monomer (gray with residues of the Pyr-redox sequence being cyan) in our data on BHM. Orange spheres indicate the phosphate groups of a simulated lipid bilayer (IMM) which was structurally aligned based on the simulation for bovine monomeric COX (PDB: 6JY3) obtained from the MemProtMD server. \textbf{C.} Comparison of the amounts of respiratory chain complexes I to IV and AIFM1 at $\sim$350 kDa representative of the COX-AIFM1\textsubscript{2} complex. Complexome profiling was performed using a standard range BN-gel (4-16\%) and a high range BN-gel (3-10\%). The average of individual values ± SD from all three conditions (untreated, PhoX and DMTMM cross-linked) is shown. \textbf{D.} Comparison of COX and AIFM1 abundances at $\sim$220 kDa, $\sim$350 kDa and $\sim$1,850 kDa in complexome profiles using a standard BN-gel (4-16\%) of BHM as prepared (untreated) and after cross-linking with PhoX and DMTMM. In \textbf{C.} and \textbf{D.}, iBAQ values of AIFM1 are divided by two to account for the dimer and the average of subunits of the respective complexes at the indicated approximate apparent masses in the migration profiles were taken as a measure for their abundance.}
        \label{fig:ch3_app_fig2}
    \end{figure*}
    \begin{figure*}[htb]
        \center
        \includegraphics[]{Chapter.3/Figures/SI_Figure3.png}
        \caption{\textbf{Structural properties of the N-terminal region of AIFM1 and cross-link distances in the COX and AIFM1 structures used for docking.} \textbf{A.} Secondary structure prediction of the N-terminal region of AIFM1 (res 55-124; sequence indicated under AA). The upper bar plot shows the confidence of residues being transmembrane residues (Score = 1) or not within a membrane (Score < 0.6). The second bar plot shows the confidence of the secondary structure prediction (1 = highest, 0 = lowest) for each residue which is indicated under PRED as letter code (C = coil, H = helix) und visualized as cartoon under CART. \textbf{B.} Detected cross-links (intra and inter domain) mapped onto the structural models of COX, AIFM1 dimer as well as the de-novo modelled N-terminal domain of AIFM1. The COX structure (green, with cross-linked subunits colored in different shades of red as indicated) was based on a previously resolved structure (PDB: 1V54) supplemented with NDUFA4 (structurally aligned based on PDB: 5Z62). Dimeric AIFM1 (orange) was modelled based on the previously resolved human homologue (PDB: 4BUR, res 128-516, 551-613). The N-terminal domain of AIFM1 (orange) (res 55-124) was generated using trRosetta. The predicted transmembrane (TM) domain is highlighted in red. \textbf{C.} Distance histogram of mapped cross-links (combination of DSSO, PhoX and DMTMM) for COX and AIFM1. AIFM1 includes distances for cross-links mapped on the AIFM1 dimer and the N-terminal region of AIFM1. For both structures, a number of cross-links are obtained within or attaching to missing regions of AIFM1 (N-terminus (55-127) and residue 517-550) and COX (N-terminus COX4l1).}
        \label{fig:ch3_app_fig3}
    \end{figure*}
    \begin{figure*}[htb]
        \center
        \includegraphics[]{Chapter.3/Figures/SI_Figure4.png}
        \caption{\textbf{Haddock cluster validation and inter cross-link distances in the COX-AIFM1\textsubscript{2} model.} \textbf{A.} Plot of HADDOCK score of the chosen clusters (note: cluster 3 was disregarded due to a positive Haddock score) as a function of their RMSD from the lowest energy structure. Circles represent individual structures of each cluster and triangles correspond to the cluster averages with standard deviation indicated by bars. \textbf{B.} Structural alignment of individual structures of cluster 7. A low average RMSD (0.942 Å) indicates high cluster precision. Different models are colored as indicated. \textbf{C.} Visualization of cross-links between COX and AIFM1 which were used for structural docking of the COX-AIFM1\textsubscript{2} complex. COX subunits are colored green, while AIFM1 protomers are colored orange. Cross-links with a C$\alpha$-C$\alpha$ distance $\leq$ 35 Å are colored in black, while cross-links with a mapped C$\alpha$-C$\alpha$ distance $\geq$ 35 Å are colored in red. Cross-linked subunits of COX are colored in different shades of red as indicated. \textbf{D.} Cross-links (combination of DSSO, PhoX and DMTMM) used for structural docking and all observed cross-links for COX-AIFM1\textsubscript{2} interaction were mapped onto the best model of the complex. For AIFM1, all obtained cross-links can be mapped onto the final structure of AIFM1, containing two AIFM1 protomers (res 128-516, 551-613) and one N-terminal region of one protomer (res 55-124).}
        \label{fig:ch3_app_fig4}
    \end{figure*}
    \begin{figure*}[htb]
        \center
        \includegraphics[]{Chapter.3/Figures/SI_Figure5.png}
        \caption{\textbf{Mapping the distance from the isoalloxazine moiety of FAD in AIFM1 to the CuA center of COX.} Cofactors were structurally aligned based on structures of AIFM1 (PDB: 4BUR) and COX (PDB: 1V54). AIFM1 protomers are orange and COX subunits are green (middle); red sticks represent FAD, black sticks represent NADH, CuA, and CuB are red (left and right). Boundaries of the IMM are indicated as gray spheres.}
        \label{fig:ch3_app_fig5}
    \end{figure*}
\end{subappendices}
\clearpage
\section*{References}
\bibliographystyle{Style_settings/bibstyle_pnas}
\bibliography{Chapter.3/chapter3_bib}
\stopthumb